<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Javier Herrero" />


<title>Twitter coverage of the Biology of Genomes 2019 conference</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lazappi's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Hashtags
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="bog2019.html">BoG19</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/jherrero/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Twitter coverage of the Biology of Genomes 2019 conference</h1>
<h4 class="author"><em>Javier Herrero</em></h4>
<h4 class="date"><em>2019-05-09 20:52:42</em></h4>

</div>


<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>An analysis of tweets from the Biology of Genomes 2019 conference. A total of 1568 tweets from 457 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1">
<h1><span class="header-section-number">2</span> Timeline</h1>
<div id="tweets-by-day" class="section level2">
<h2><span class="header-section-number">2.1</span> Tweets by day</h2>
<p><img src="figures/bog2019/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2">
<h2><span class="header-section-number">2.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2019-05-07 - 2019-05-09 in the America/New_York timezone.</p>
<p><img src="figures/bog2019/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1">
<h1><span class="header-section-number">3</span> Users</h1>
<div id="top-tweeters" class="section level2">
<h2><span class="header-section-number">3.1</span> Top tweeters</h2>
<p><img src="figures/bog2019/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-original-tweeters" class="section level2">
<h2><span class="header-section-number">3.2</span> Top original tweeters</h2>
<p><img src="figures/bog2019/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-proportion" class="section level2">
<h2><span class="header-section-number">3.3</span> Retweet proportion</h2>
<p><img src="figures/bog2019/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-time" class="section level2">
<h2><span class="header-section-number">3.4</span> Top tweeters by time</h2>
<p><img src="figures/bog2019/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2">
<h2><span class="header-section-number">3.5</span> Top tweeters by day</h2>
<div id="all-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.1</span> All tweets</h3>
<div id="day-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/bog2019/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/bog2019/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/bog2019/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.2</span> Original tweets</h3>
<div id="day-1-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="sources" class="section level2">
<h2><span class="header-section-number">3.6</span> Sources</h2>
<p><img src="figures/bog2019/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="networks" class="section level1">
<h1><span class="header-section-number">4</span> Networks</h1>
<div id="replies" class="section level2">
<h2><span class="header-section-number">4.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by page rank.</p>
<p><img src="figures/bog2019/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2">
<h2><span class="header-section-number">4.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Node colour, label size and alpha adjusted according to PageRank score.</p>
<p><img src="figures/bog2019/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets" class="section level1">
<h1><span class="header-section-number">5</span> Retweets</h1>
<div id="retweet-proportion-1" class="section level2">
<h2><span class="header-section-number">5.1</span> Retweet proportion</h2>
<p><img src="figures/bog2019/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-count" class="section level2">
<h2><span class="header-section-number">5.2</span> Retweet count</h2>
<p><img src="figures/bog2019/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-retweets" class="section level2">
<h2><span class="header-section-number">5.3</span> Top retweets</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Just in time for #bog19, <span class="citation">@genomeresearch</span> posts early access version (today) of our paper where we found 2250 bacterial genomes that had human contamination, plus 3,437 “proteins” that are erroneous translations of these human contaminants <a href="https://t.co/koQRkmLRPw" class="uri">https://t.co/koQRkmLRPw</a>
</td>
<td style="text-align:right;">
38
</td>
</tr>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Peak east coast techno-hipster cryptocurrency Americana cannabis genome #bog19 <a href="https://t.co/x4hOYHdeGe" class="uri">https://t.co/x4hOYHdeGe</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
athmapai
</td>
<td style="text-align:left;">
Recent or soon to be grads: looking for a postdoc in gene regulation, RNA, and genomics? I’d love to chat! Come find me at #BoG19! I’ll be presenting poster #185 on Thursday <a href="https://t.co/FBsPTVH0Tn" class="uri">https://t.co/FBsPTVH0Tn</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
aaronquinlan
</td>
<td style="text-align:left;">
Really fun talk from <span class="citation">@BeEngelhardt</span> on a Bayesian approach for optimizing experimental design to maximize cell type discovery from single-cell sequencing experiments. Calling it GT-TS (Good-Toulmin like estimator via Thompson sampling). Paper here: <a href="https://t.co/uWm7ySP4lG" class="uri">https://t.co/uWm7ySP4lG</a> #bog19
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Olga Sigalova, Nice to see discussion of Broad vs narrow promoters (once identified by CAGE back in Fantom3), whereas broad promoters tend to be housekeeping and narrow drive expression of more specific genes; reviewed in: <a href="https://t.co/0IQWlNnKMn" class="uri">https://t.co/0IQWlNnKMn</a> #bog19
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Transatlantic wifi and I can feel the genome tribes gathering for #bog19 - a little history for those not acquainted with this meeting
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Now Mike Schatz <span class="citation">@mike_schatz</span> on assembling many tomato 🍅 genomes 🧬 and structural variants (fellow <span class="citation">@GenomicsAtJHU</span> member)</p>
<p>Uses <span class="citation">@nanopore</span> and HiC data, then RaGOO software</p>
<p>Code 💻: <a href="https://t.co/ljO0NzR0FB" class="uri">https://t.co/ljO0NzR0FB</a> 📜: <a href="https://t.co/ge0PJLQCix" class="uri">https://t.co/ge0PJLQCix</a> 🖇️: <a href="https://t.co/QAmuwXDF4E" class="uri">https://t.co/QAmuwXDF4E</a></p>
#BoG19
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
PLOSBiology
</td>
<td style="text-align:left;">
If you’ve been “Scooped,” remember <span class="citation">@PLOSBiology</span>’s anti-scooping policy; come to us within 6 months of “scooping” and we’ll consider it regardless. Read our policy here: <a href="https://t.co/hMd2vrXPqU" class="uri">https://t.co/hMd2vrXPqU</a> or if you’re at #BoG19 you can just talk to <span class="citation">@RoliRoberts</span>… <a href="https://t.co/leZMMON2rh" class="uri">https://t.co/leZMMON2rh</a>
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
#bog19 the first BOG without the portrait of Watson. Finally. <a href="https://t.co/i42jfyMKdt" class="uri">https://t.co/i42jfyMKdt</a>
</td>
<td style="text-align:right;">
9
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="favourites" class="section level1">
<h1><span class="header-section-number">6</span> Favourites</h1>
<div id="favourite-proportion" class="section level2">
<h2><span class="header-section-number">6.1</span> Favourite proportion</h2>
<p><img src="figures/bog2019/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="favourite-count" class="section level2">
<h2><span class="header-section-number">6.2</span> Favourite count</h2>
<p><img src="figures/bog2019/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-favourites" class="section level2">
<h2><span class="header-section-number">6.3</span> Top favourites</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
200
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Peak east coast techno-hipster cryptocurrency Americana cannabis genome #bog19 <a href="https://t.co/x4hOYHdeGe" class="uri">https://t.co/x4hOYHdeGe</a>
</td>
<td style="text-align:right;">
77
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Just in time for #bog19, <span class="citation">@genomeresearch</span> posts early access version (today) of our paper where we found 2250 bacterial genomes that had human contamination, plus 3,437 “proteins” that are erroneous translations of these human contaminants <a href="https://t.co/koQRkmLRPw" class="uri">https://t.co/koQRkmLRPw</a>
</td>
<td style="text-align:right;">
70
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Love it that the genome scaffolders used by <span class="citation">@mike_schatz</span> for the tomato genome are called SALSA2 and RaGOO. Coincidence? #BoG19
</td>
<td style="text-align:right;">
60
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
#bog19 the first BOG without the portrait of Watson. Finally. <a href="https://t.co/i42jfyMKdt" class="uri">https://t.co/i42jfyMKdt</a>
</td>
<td style="text-align:right;">
58
</td>
</tr>
<tr>
<td style="text-align:left;">
aaronquinlan
</td>
<td style="text-align:left;">
Really fun talk from <span class="citation">@BeEngelhardt</span> on a Bayesian approach for optimizing experimental design to maximize cell type discovery from single-cell sequencing experiments. Calling it GT-TS (Good-Toulmin like estimator via Thompson sampling). Paper here: <a href="https://t.co/uWm7ySP4lG" class="uri">https://t.co/uWm7ySP4lG</a> #bog19
</td>
<td style="text-align:right;">
48
</td>
</tr>
<tr>
<td style="text-align:left;">
lukejostins
</td>
<td style="text-align:left;">
At Biology of Genomes this week. A bit jet lagged for live tweeting right now, but jesus there are some unsettlingly gifted graduate students giving talks tonight #BoG19
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
kajawasik
</td>
<td style="text-align:left;">
Whenever I’m back at <span class="citation">@CSHL</span> I realize how much I miss this place. It really made me a scientist and taught me to enjoy the small things in life like wine and cheese on President’s lawn.. #bog19 <a href="https://t.co/efXS121Mh9" class="uri">https://t.co/efXS121Mh9</a>
</td>
<td style="text-align:right;">
44
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Transatlantic wifi and I can feel the genome tribes gathering for #bog19 - a little history for those not acquainted with this meeting
</td>
<td style="text-align:right;">
42
</td>
</tr>
<tr>
<td style="text-align:left;">
anderson_carl
</td>
<td style="text-align:left;">
Great #bog19 quote from <span class="citation">@hilsomartin</span>: Just because the phenotyping is expensive doesn’t change the power calculations!
</td>
<td style="text-align:right;">
42
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="quotes" class="section level1">
<h1><span class="header-section-number">7</span> Quotes</h1>
<div id="quote-proportion" class="section level2">
<h2><span class="header-section-number">7.1</span> Quote proportion</h2>
<p><img src="figures/bog2019/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="quote-count" class="section level2">
<h2><span class="header-section-number">7.2</span> Quote count</h2>
<p><img src="figures/bog2019/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-quotes" class="section level2">
<h2><span class="header-section-number">7.3</span> Top quotes</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
AnnaCuomo92
</td>
<td style="text-align:left;">
Great Illustration and great meeting so far! #BoG19 <a href="https://t.co/MbRJgVUS5E" class="uri">https://t.co/MbRJgVUS5E</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
Awesomics
</td>
<td style="text-align:left;">
The Biology of Genomes (#BoG19) now underway <a href="https://t.co/uJpygePI9e" class="uri">https://t.co/uJpygePI9e</a>. Access live streamed talks, abstract PDF, and video archive (for 6 weeks) at <a href="https://t.co/L8JiHY0BFH" class="uri">https://t.co/L8JiHY0BFH</a> and enter “99” in the ‘Payment Authorized’ field. Cost=$99. Thanks <span class="citation">@cshlmeetings</span>! <a href="https://t.co/a4zUxtCduV" class="uri">https://t.co/a4zUxtCduV</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="media" class="section level1">
<h1><span class="header-section-number">8</span> Media</h1>
<div id="media-count" class="section level2">
<h2><span class="header-section-number">8.1</span> Media count</h2>
<p><img src="figures/bog2019/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-media" class="section level2">
<h2><span class="header-section-number">8.2</span> Top media</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
200
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Peak east coast techno-hipster cryptocurrency Americana cannabis genome #bog19 <a href="https://t.co/x4hOYHdeGe" class="uri">https://t.co/x4hOYHdeGe</a>
</td>
<td style="text-align:right;">
77
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
#bog19 the first BOG without the portrait of Watson. Finally. <a href="https://t.co/i42jfyMKdt" class="uri">https://t.co/i42jfyMKdt</a>
</td>
<td style="text-align:right;">
58
</td>
</tr>
<tr>
<td style="text-align:left;">
kajawasik
</td>
<td style="text-align:left;">
Whenever I’m back at <span class="citation">@CSHL</span> I realize how much I miss this place. It really made me a scientist and taught me to enjoy the small things in life like wine and cheese on President’s lawn.. #bog19 <a href="https://t.co/efXS121Mh9" class="uri">https://t.co/efXS121Mh9</a>
</td>
<td style="text-align:right;">
44
</td>
</tr>
<tr>
<td style="text-align:left;">
ChrisOmics
</td>
<td style="text-align:left;">
My girl <span class="citation">@ruthie_johnson</span> talking about her new method BEAVR estimating proportions of causal SNPs in a region at #BoG19. <a href="https://t.co/KwbHQerO87" class="uri">https://t.co/KwbHQerO87</a>
</td>
<td style="text-align:right;">
32
</td>
</tr>
<tr>
<td style="text-align:left;">
PLOSBiology
</td>
<td style="text-align:left;">
If you’ve been “Scooped,” remember <span class="citation">@PLOSBiology</span>’s anti-scooping policy; come to us within 6 months of “scooping” and we’ll consider it regardless. Read our policy here: <a href="https://t.co/hMd2vrXPqU" class="uri">https://t.co/hMd2vrXPqU</a> or if you’re at #BoG19 you can just talk to <span class="citation">@RoliRoberts</span>… <a href="https://t.co/leZMMON2rh" class="uri">https://t.co/leZMMON2rh</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
JedMSP
</td>
<td style="text-align:left;">
<p>Come see me at poster 46 to talk about this hell site with me</p>
#BoG19 <a href="https://t.co/5PQZtqyFzx" class="uri">https://t.co/5PQZtqyFzx</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
bpasaniuc
</td>
<td style="text-align:left;">
Great job from <span class="citation">@ruthie_johnson</span> presenting the regional polygenic estimation work at the #BoG19 ! <a href="https://t.co/WTwsxEDLdQ" class="uri">https://t.co/WTwsxEDLdQ</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
Taking a detour to the Brooklyn Botanic Garden on the way to #bog19. I’m so excited to be back!! <a href="https://t.co/CiObVNKySH" class="uri">https://t.co/CiObVNKySH</a>
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
wbmei
</td>
<td style="text-align:left;">
Arrive the beautiful cold spring harbor campus and exited about learning more genetics and genomics in the next couple days #BoG19 <a href="https://t.co/ak2apnJHXD" class="uri">https://t.co/ak2apnJHXD</a>
</td>
<td style="text-align:right;">
27
</td>
</tr>
</tbody>
</table>
<div id="most-liked-media-image" class="section level3">
<h3><span class="header-section-number">8.2.1</span> Most liked media image</h3>
<div class="figure">
<img src="http://pbs.twimg.com/media/D5-M0UeU8AAd3BC.jpg" />

</div>
</div>
</div>
</div>
<div id="tweet-text" class="section level1">
<h1><span class="header-section-number">9</span> Tweet text</h1>
<div id="word-cloud" class="section level2">
<h2><span class="header-section-number">9.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/bog2019/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2">
<h2><span class="header-section-number">9.2</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 5 times.</p>
<p><img src="figures/bog2019/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2">
<h2><span class="header-section-number">9.3</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/bog2019/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset">
<h3><span class="header-section-number">9.3.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered">
<h4>Topic 1</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
majbvc
</td>
<td style="text-align:left;">
El <span class="citation">@centrodemocratico</span> buscará un candidato distinto a Ángela Garzón para enfrentar las elecciones en Bogotá. Será…??? O será otra salida en falso…?? #losleo <span class="citation">@AlvaroUribeVel</span> <span class="citation">@diegomolanovega</span> <span class="citation">@SamuelHoyosM</span> #FelizJueves #BoG19 #filbo2019 #video <span class="citation">@ChurchillBogota</span> <span class="citation">@MiguelUribeT</span> <a href="https://t.co/GUJ8jaznOy" class="uri">https://t.co/GUJ8jaznOy</a>
</td>
<td style="text-align:right;">
0.9944930
</td>
</tr>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
.<span class="citation">@nonfiniteloop</span> — Can we detect post zygotic variations. Re-purpose GTEx data. Develop method to detect mutations from bulk RNA-seq and validate in Sun exposed/unexposed skin. More mutations in UV exposed skin, increase in UV-induced mut. signature #BoG19
</td>
<td style="text-align:right;">
0.9944930
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Rockweiler/3. Mutations are tissue specific (example: C-&amp;gt;T in skin). Brain less mutations. Age/timing? Shared mutation is more ancient -&amp;gt; Surprise: 80% of mutations happen between zygote and gastrula! (surprise: in so few cell cycles!) Do not miss cell specific ones? #bog19
</td>
<td style="text-align:right;">
0.9936186
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Rockweiler/2: measuring skin exposed/not to UVs showing more skin mutation in caucasians if compared to African American donors. identify 79 mutations/donor, 4/tissues, 7% overlap, &amp;gt;82-88% validation rate. Mutation rate very variable across tissues and samples. #bog19
</td>
<td style="text-align:right;">
0.9936186
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Nicole Rockweiler, whole body maps postzygotic mutations in humans (= after fertilization through lifespan). The genome is not the same in all cells or in twins! Discuss: where, when? Repurposing GTEx data, 17K RNA-seq libraries, reconstruction of genome data from RNA seq. #bog19
</td>
<td style="text-align:right;">
0.9936186
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
NW - This is a relatively new field. 3 questions - can we detect these mutations. Repurposing GTex 964 postmortem donors for post zygotic mutation. Use sun exposed vs non sun exposed skin - see increase in mutation burden in sun exposed skin in caucasians #bog19
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>NB <span class="citation">@nonfiniteloop</span> found the lowest mutation burden in the brain, and highest in liver followed by EBV (missed info here) &amp;amp; esophagus</p>
<p>NB’s main questions: Can we detect postzygotic mutations? where are they? &amp;amp; when they occur?</p>
<p>On Q3: some tissues increase 🧬mut with age</p>
#BoG19
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
OS — broad promoters &amp;lt;=&amp;gt; housekeeping genes ; narrow promoters and low variation =&amp;gt; GO: developments, regulation of transcription; narrow promoters and high variation =&amp;gt; Metabolic processes (lots of TAT box as well) #BoG19
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
skathire
</td>
<td style="text-align:left;">
<p>Really beautiful talk by <span class="citation">@nonfiniteloop</span> on post-zygotic mutations in GTeX</p>
<p>What looks to be first survey of somatic mutations across many tissues</p>
<p>Tissue with highest rate - liver Lowest rate - brain</p>
<p>Rate in most tissues is age dependent but not all</p>
<p>Cc: <span class="citation">@jaiswalmdphd</span></p>
#BoG19
</td>
<td style="text-align:right;">
0.9930683
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Andrew Jaffe (partnering with companies in BrainSeq consortium). In this talk focus on hippocampus, laser capture, bulk RNAseq from LCM of dentate gyrus (DG). Then DG specific eQTL, finding eQTLs associated with schizophrenia. #bog19
</td>
<td style="text-align:right;">
0.9930683
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered">
<h4>Topic 2</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Fiona Calvert; regulatory map of human primary microglia (~resident macrophages of CNS). Why important? -&amp;gt; genetics data points at association with Alzheimers. Single cell sequencing (15 patients) -&amp;gt; association with neurological disorders. Here 141 neurosurgery patients. #BoG19
</td>
<td style="text-align:right;">
0.9936186
</td>
</tr>
<tr>
<td style="text-align:left;">
cshlmeetings
</td>
<td style="text-align:left;">
Unable to make it to next week’s #BoG19? Live stream it! Each $99 subscription includes live stream of talks, PDF of the abstract book, and six-week access to #BoG19’s video archive. Register online: <a href="https://t.co/fMcgEzbvys" class="uri">https://t.co/fMcgEzbvys</a> and enter “99” in the ‘Payment Authorized’ field. <a href="https://t.co/AHcQxt1Sg6" class="uri">https://t.co/AHcQxt1Sg6</a>
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Wu on personal cancer vaccines - increasing the response to more than 30%. 1) Antitumor immunosensitization, 2) preventing immunosuppression. -&amp;gt; therapeutic vaccines. Nice image, cancer vaccine is the steering wheel (PD-1 is the brake). How: use cancer-mutated peptides #BoG19
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Final speaker for this #BoG19 session: Yi Qiao from the Marth Lab <span class="citation">@MarthGabor</span> on studying heterogeneity in cancer somatic subclones</p>
<p>Earlier methods: PyClone, SciClone, SubcloneSeeker</p>
<p>Presenting today is a “Dream come true” ^^</p>
<a href="https://t.co/tb1tqYpVet" class="uri">https://t.co/tb1tqYpVet</a> <a href="https://t.co/067j2RdgLU" class="uri">https://t.co/067j2RdgLU</a>
</td>
<td style="text-align:right;">
0.9927560
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
MS: ~900 varieties of tomato already sequenced with short reads, has SNP-based phylogeny. Based on these data, recommend using consensus caller for SV in these to improve PF rate, though may still have systematic bias due to aligning. #BoG19
</td>
<td style="text-align:right;">
0.9927560
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Fiona <span class="citation">@ficalvert</span> worked with single cell data from 141 neurosurgery patients split into 5 groups: tumor, trauma, hydrocephalus, hemorrhage, control</p>
<p>Used Seurat for clustering the #scRNAseq data (not cited in slide 🧐😑)</p>
<p>They had brain and blood samples from each indiv.</p>
#BoG19
</td>
<td style="text-align:right;">
0.9927560
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
TR: Local adaptation with LMM - association between genotypes and climate variables. Temperature, precipitation, and elevation for every sq meter of U.S (from PRISM). k-mean clustering of the three variables, to divide US into 9 zones and link to animal birthplace. #BoG19
</td>
<td style="text-align:right;">
0.9927560
</td>
</tr>
<tr>
<td style="text-align:left;">
cshlmeetings
</td>
<td style="text-align:left;">
It is indeed a great alternative for those unable to make it here in person. In addition to the live streamed talks are a PDF of the abstract book and 6-week access to #BoG19’s video archive. Register at <a href="https://t.co/fMcgEzbvys" class="uri">https://t.co/fMcgEzbvys</a> and enter “99” in the ‘Payment Authorized’ field. <a href="https://t.co/gZRGpanOOt" class="uri">https://t.co/gZRGpanOOt</a>
</td>
<td style="text-align:right;">
0.9924141
</td>
</tr>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
Daniel S. Kim — In sillico mutagenesis to infer synergestic interactions btw motifs within a locus. Build a pairwise full matrix of synergies in skin differentiation to define a “grammar” of motifs (only pairs of motifs) #BoG19
</td>
<td style="text-align:right;">
0.9924141
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Next Fiona Calvert <span class="citation">@ficalvert</span> from Daniel Gaffney’s group <span class="citation">@danjgaffney</span> at <span class="citation">@wellcometrust</span> <span class="citation">@sangerinstitute</span> (2nd in a row 😮)</p>
<p>(final year PhD student on the job market)</p>
<p>Fiona is interested in microglia, starts comparing a human one vs one that is iPSC-derived</p>
#BoG19
</td>
<td style="text-align:right;">
0.9924141
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-3" class="section level4 unnumbered">
<h4>Topic 3</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>SMAC-seq by Georgi Marinov and team re-captures the peaks by DNA-seq peak but also overlaps H4S37C chemical nucleosome mapping peaks</p>
<p>Their profiles are nicely phased with Cse4 ChIP-seq as well</p>
<p>(I don’t know what Cse4 nor H4S37C do! Tweet and later revisit? :P)</p>
#bog19
</td>
<td style="text-align:right;">
0.9938622
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Hannah Currant, on UK Biobank, 500K individuals, 96M variants: focus on retinal phenotype. Retina inspected for physical differences (example: thickness, as “endophenotype”), which is connected with diseases/biology. ~31K GWAS- identify loci associated with retina problems #bog19
</td>
<td style="text-align:right;">
0.9936186
</td>
</tr>
<tr>
<td style="text-align:left;">
obahcall
</td>
<td style="text-align:left;">
Appreciation for our wonderful #BoG19 organizers, who are also illustrious scientists celebrating recent well deserved recognitions!<br />
Matt Hurles <span class="citation">@mehurles</span> elected FRS <span class="citation">@royalsociety</span> Elaine Ostrander elected <span class="citation">@theNASciences</span> Jay Shendure <span class="citation">@JShendure</span> Lounsbery award <span class="citation">@theNASciences</span>
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Mikkel <span class="citation">@mikkelschierup</span> is talking about Skov et al, bioRxiv, 2018 <a href="https://t.co/modrYeXWWz" class="uri">https://t.co/modrYeXWWz</a></p>
<p>They can use this to derive mean archaic segmenting lengths (SGDP) &amp;amp; the total amount of archaic sequence</p>
<p>Then they explored how the segments are shared by West Eurasia &amp;amp; East Asia</p>
#BoG19 <a href="https://t.co/XSh7Ah1kmI" class="uri">https://t.co/XSh7Ah1kmI</a>
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Next up Anat Kreimer <span class="citation">@anatkreimer</span> from Nir Yosef’s lab on using massively parallel reporter assays (MPRAs) for high-throughput enhancer validation (uses a lentivirus that is transducted, then followed by RNA and DNA seq)</p>
<p>Related pre-print: <a href="https://t.co/UeaaDYWm6U" class="uri">https://t.co/UeaaDYWm6U</a></p>
#bog19
</td>
<td style="text-align:right;">
0.9930683
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
DK - first mention of deep neural networks. Does some nice transfer learning from epigenome/ENCODE deep neural networks. Pretty good model regression from DNA to activity. (Deep Neural Networks for the win!) #bog19
</td>
<td style="text-align:right;">
0.9927560
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
TR: nearly 1M beef cattle animal genotyped. Now have large accurately imputed dataset, temporally distributed tracing back to specimen from 70s. Recent history well understood (pedigree to early 1900s), strong and recent selection. #BoG19
</td>
<td style="text-align:right;">
0.9927560
</td>
</tr>
<tr>
<td style="text-align:left;">
NeBanovich
</td>
<td style="text-align:left;">
YQ: Missing link is tying genomic subclocal structure to transcriptional structure - can now use scRNA-seq and bulk RNAseq to tie these data together. However not trivial, largely due to low coverage in scRNA-seq data #bog19
</td>
<td style="text-align:right;">
0.9927560
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
HC: Clusters SNPs in a 24-dimensional space from the OCT data, and fall out into 3 quite separate clusters. One of the clusters with MR support a GCIPL &amp;lt;=&amp;gt; Glaucoma link (which is consistent with epidemiology / ophthalmology) #bog19. She handles the questions well
</td>
<td style="text-align:right;">
0.9927560
</td>
</tr>
<tr>
<td style="text-align:left;">
abarysh
</td>
<td style="text-align:left;">
Just posted an ad for a compbio position in my group (+ other Calico opportunities) on the board next to CSHL meetings office (Grace Auditorium, downstairs). #BOG19 folks on the job market: check it out and/or reach out to me for a chat–happy to give all the details in person!
</td>
<td style="text-align:right;">
0.9924141
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered">
<h4>Topic 4</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Julie <span class="citation">@LiliNYC11</span> focused on dopaminergic neurons from substantia nigra, related to Parkinson’s disease</p>
<p>Built HipSci: nice logo!</p>
<p>Ran into some problems a few times and had to start from a frozen pool. Ideally, start from a fresh sample</p>
Cells followed expected patt day 11 #BoG19
</td>
<td style="text-align:right;">
0.9938622
</td>
</tr>
<tr>
<td style="text-align:left;">
nanopore
</td>
<td style="text-align:left;">
Will you be at Biology of Genomes next week? Register for our free lunch &amp;amp; seminar on Thursday 9th May. Steven Salzberg is giving a talk on ‘The Redwood Genome Project: sequencing and assembling the largest trees in the world’ join us here: <a href="https://t.co/gq8U4SXRh8" class="uri">https://t.co/gq8U4SXRh8</a> #bog19 <a href="https://t.co/GTbHxxK06n" class="uri">https://t.co/GTbHxxK06n</a>
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
kasper_lage
</td>
<td style="text-align:left;">
Thanks for the shoutout based on my #BoG19 talk about our <span class="citation">@StanleyCenter</span> <span class="citation">@broadinstitute</span> project to map protein networks of schizophrenia risk genes in human neurons. A great collaboration with <span class="citation">@EgganLab</span>. <span class="citation">@mjapkim</span> in the lab developed the Genoppi platform we use to QC data: <a href="https://t.co/7XGF4LhQfz" class="uri">https://t.co/7XGF4LhQfz</a>
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Now Robin Friedman <span class="citation">@RCFbio</span> from Ohana Biosciences on haploid-biased gene expression in mammalian spermatogenesis and its links to selected sweeps</p>
<p>“I spend most of my time working on sperm biology” lead to lots of 😆🤣 ^^ (we all love jokes hehe)</p>
<p><a href="https://t.co/X1WXW2BE6Z" class="uri">https://t.co/X1WXW2BE6Z</a></p>
#BoG19
</td>
<td style="text-align:right;">
0.9930683
</td>
</tr>
<tr>
<td style="text-align:left;">
sorrywm
</td>
<td style="text-align:left;">
Interesting finding from Yi Qiao’s talk: “mis-classified” sub-clones from breast cancer longitudinal dataset using scBayes (actually pre-treatment but classified as post-treatment) show that doxorubicin treatment produces selection on standing variation #BoG19
</td>
<td style="text-align:right;">
0.9930683
</td>
</tr>
<tr>
<td style="text-align:left;">
ficalvert
</td>
<td style="text-align:left;">
Thought I’d beaten jet lag by forcing myself to stay up late on my first night in New York. My 5am wake up this morning was a brutal reminder that you can never beat jet lag! Excited to go explore the city after using my early start to work on my #BoG19 slides!
</td>
<td style="text-align:right;">
0.9927560
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
RF: punchline = sperm-level natural selection via transmission ratio distortion. Diploid spermatocytes -&amp;gt; haploid spermatids -&amp;gt; haploid spermatozoa. But haploid spermatids are still transcribing from the haploid genome. #BoG19
</td>
<td style="text-align:right;">
0.9927560
</td>
</tr>
<tr>
<td style="text-align:left;">
Namaslay2
</td>
<td style="text-align:left;">
#giveaway #GiveawayAlert #kpop #kpopgiveaway #BTS #EXO #BLACKPINKinBANGKOK #blackpink #twice #BLINKSelcaDay #ARMYSelcaDay #EXOLSelcaDay #ONCE #MetGala #BillboardNews #Hot100 #RoseBowl #io19 #WednesdayWisdom #BoG19 #RamadanMubarak #WednesdayMotivation #LiverpoolFC <a href="https://t.co/YrM9PCQggA" class="uri">https://t.co/YrM9PCQggA</a>
</td>
<td style="text-align:right;">
0.9924141
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
RF: Talks us about spermatid biology which form syncetium between gametes (!) - freely sharing the mRNA between spermatids but some genes can be selfish - this is the explanation for the Mouse T complex genetics - lead to extreme transmission distortion. #bog19
</td>
<td style="text-align:right;">
0.9924141
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Next, Hannah Currant <span class="citation">@hannahcurrant1</span> (from <span class="citation">@ewanbirney</span>’s group at <span class="citation">@emblebi</span> ) on using the UK Biobank to go from GWAS to phenotype, focusing on the retina</p>
<p>They have image segmentation phenotype data using OCT optical conference tomography</p>
Related: <a href="https://t.co/dQx8VmKsW7" class="uri">https://t.co/dQx8VmKsW7</a> #BoG19
</td>
<td style="text-align:right;">
0.9924141
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered">
<h4>Topic 5</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Sanders: Structural variations in single cells. Developed a method for single cell genome sequencing which maintains strand information, taking advantage of BdrU incorporation -&amp;gt; digestion of one strand -&amp;gt; sequencing the remaining strand. Measures depth, strand, phase. #BoG19
</td>
<td style="text-align:right;">
0.9944930
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Javier Mendoza-Revilla, history of south-west Pacific migrations. Only a recent 5.5 k years ago went to far Oceania, then replaced by Papuan ancestry 2.5K years ago. Complex -&amp;gt; sequences 317 WHG. PCA separates nicely for geographical regions of donors. #BoG19
</td>
<td style="text-align:right;">
0.9940879
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
JMR: Settlement of Near / Remote Oceania - modern humans arrived in Near Oceania ~40-65kya, but did not extend beyond Soloman island. Austronesian expansion from Taiwan ~ 5.5 kya; Lapita people arrived in Vanuatu ~3kya; mirgration from Bismarck Archipelago ~2.5kya. #BoG19
</td>
<td style="text-align:right;">
0.9938622
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Elise <span class="citation">@EliseScience</span> thinks that the + and - correlated eQTLs with aFC could be due to transcription factor binding levels</p>
<p>Correlated aFC with TF expression in each tissue (used ENCODE data)</p>
<p>5.6% of tested TF-eQTLs pairs are corr with 1 TF 41% of are corr with &amp;gt; 1TF</p>
#BoG19
</td>
<td style="text-align:right;">
0.9938622
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
KL: Channel island foxes, population size ~300-2000; S. Nicolas foxes severely reduced diversity across the genome, &amp;gt; 10x reduction in heterozygosity. Adding more samples, including samples from the past (1929), and low diversity has been observed even in old samples. #BoG19
</td>
<td style="text-align:right;">
0.9936186
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Wu/2 capitalize on exome sequencing, predict candidate new antigens. POC: Melanoma, predicting antigens, manufacturing antigens, given to patients with adjuvant to stimulate immune memory. Other study-glioblastoma. Combination with anti- PD-1. Successful promising so far. #BoG19
</td>
<td style="text-align:right;">
0.9936186
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
KL: Flipping to Isle Royale population, where they have been recently isolated - very bumpy hetreozygosity plot, and derived deleterious alleles, and not a healthy population. Suggests the Island fox population inbreed with strong deleterious alleles removed #bog19
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
MS: greedily identify the most diverse species from short-read data to identify candidates for long-read sequencing. PromethION improved sequencing throughput dramatically. Read length ~10-100kb length, rarily &amp;gt; 1 Mb. #BoG19
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
aaronquinlan
</td>
<td style="text-align:left;">
Really fun talk from <span class="citation">@BeEngelhardt</span> on a Bayesian approach for optimizing experimental design to maximize cell type discovery from single-cell sequencing experiments. Calling it GT-TS (Good-Toulmin like estimator via Thompson sampling). Paper here: <a href="https://t.co/uWm7ySP4lG" class="uri">https://t.co/uWm7ySP4lG</a> #bog19
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
MS: Need to bring some new technology to capture SVs. there are 50 varieties with short reads from 10 main clades. Long Reads from <span class="citation">@nanopore</span> - start at MinION+GridION - 5/10 flowcells per genome - but then PromethION really changed the game. Best run 140GB flow cell #BoG19
</td>
<td style="text-align:right;">
0.9930683
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered">
<h4>Topic 6</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
JH: chromatin-modifying genes (CMBs) - diverse set of genes that bind chromatin and regulate cellular processes (remodelling or modification). In DDD project (13.6k patient with undiagnosed DD) de novo variants explain ~50% of risk, enriched in CMG set. #BoG19
</td>
<td style="text-align:right;">
0.9944930
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
RJ: Causal effect size distribution parameterized by effect size (beta) and number of causal SNPs (k) (Zhang et al. NG 2018). Want to know how is heritability divided - centered on 1 SNP or distributed among multiple SNPs, has implication to effect distr at regional level. #BoG19
</td>
<td style="text-align:right;">
0.9938622
</td>
</tr>
<tr>
<td style="text-align:left;">
rnomics
</td>
<td style="text-align:left;">
<p>Ewan Birney on Twitter: “Sek takes us through the construction of polygenic risk score - from MI GWAS, selected best score from 20 with first round of UK BioBank, then test on remainder. PRS is not correlated to the ACC/AHA risk score #BoG19” <a href="https://t.co/iN0JYt345f" class="uri">https://t.co/iN0JYt345f</a></p>
twitte…
</td>
<td style="text-align:right;">
0.9938622
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Coronary artery disease is the leading global cause of mortality</p>
<p>Current risk assessment of heart attack reviewed by Khera et al <a href="https://t.co/7BJCbpN2xv" class="uri">https://t.co/7BJCbpN2xv</a> based on non-🧬metrics</p>
but <span class="citation">@skathire</span> has now shown that polygenic risk score is the first risk factor for this disease #BoG19 <a href="https://t.co/UuHPODF7cq" class="uri">https://t.co/UuHPODF7cq</a>
</td>
<td style="text-align:right;">
0.9936186
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
SK: Hypothesis = polygenic scores including genome-wide set of SNPs can identify more individuals at risk of MI. 60k cases vs. 120k controls, GWAS with imputation data to construct PRS, and then validated PRS in UK Biobank. #BoG19
</td>
<td style="text-align:right;">
0.9936186
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Peter <span class="citation">@genemodeller</span> on types of joint analysis:</p>
<ul>
<li>meta-analysis</li>
<li>fine-mapping</li>
<li>PheWAS-analysis</li>
</ul>
<p>You could also cluster GWAS results. Did so using a bayesian approach (would make <span class="citation">@BeEngelhardt</span> proud ^^) based on a Dirichlet prior</p>
<p>Can analyze simultaneously &amp;gt;1k GWAS</p>
#BoG19
</td>
<td style="text-align:right;">
0.9936186
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
RJ: BEAVR estimates local polygenicity from GWAS summary statistics. Use GWAS “noisy” effect size estimates to infer underlying effect size distr. Can estimate # of causal SNPs across varying level of heritability. #BoG19
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Elise <span class="citation">@EliseScience</span> measures eQTL effects using the following statistic:</p>
<p>aFC = log2( ALT allele expr / REF allele expr)</p>
<p>Should be stable across genes</p>
<p>aFC is diff in some tissues, 10% of eQTLs had aFC highly correlated with eGene expr (either + or -)</p>
#BoG19
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Peter <span class="citation">@genemodeller</span> wants to build polygenic risk scores for many traits using their fine mapping approach</p>
<p>He looked at the top 3, lowest 3 and middle 20% risk scores with the weights applied to <span class="citation">@uk_biobank</span> data. The results can inform policies for when to do screenings</p>
#BoG19
</td>
<td style="text-align:right;">
0.9933548
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Day 3 talk 1: Hilary Martin <span class="citation">@hilsomartin</span> at <span class="citation">@wellcometrust</span> (I believe she works with <span class="citation">@ee_davenport</span>, no?, well, and many more people ^^) on deciphering developmental disorders</p>
<p>Male: female ratio = 1.4 in DDD (protective effect for females) chrX enriched for known DD genes</p>
#BoG19
</td>
<td style="text-align:right;">
0.9930683
</td>
</tr>
</tbody>
</table>
<hr />
</div>
</div>
</div>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1>Session info</h1>
<pre><code>## R version 3.5.1 (2018-07-02)
## Platform: x86_64-apple-darwin15.6.0 (64-bit)
## Running under: macOS  10.14.4
## 
## Matrix products: default
## BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
## LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] en_GB.UTF-8/en_GB.UTF-8/en_GB.UTF-8/C/en_GB.UTF-8/en_GB.UTF-8
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] bindrcpp_0.2.2     here_0.1           knitr_1.20        
##  [4] viridis_0.5.1      viridisLite_0.3.0  ggraph_1.0.2      
##  [7] wordcloud_2.6      RColorBrewer_1.1-2 ggrepel_0.8.0     
## [10] ggplot2_3.0.0      igraph_1.2.2       topicmodels_0.2-8 
## [13] tidytext_0.2.0     forcats_0.3.0      lubridate_1.7.4   
## [16] tidyr_0.8.1        dplyr_0.7.6        rtweet_0.6.8      
## 
## loaded via a namespace (and not attached):
##  [1] Rcpp_0.12.19      lattice_0.20-35   utf8_1.1.4       
##  [4] assertthat_0.2.0  rprojroot_1.3-2   digest_0.6.18    
##  [7] ggforce_0.1.3     slam_0.1-45       R6_2.3.0         
## [10] plyr_1.8.4        backports_1.1.2   stats4_3.5.1     
## [13] evaluate_0.12     highr_0.7         httr_1.3.1       
## [16] pillar_1.3.0      rlang_0.2.2       lazyeval_0.2.1   
## [19] curl_3.2          Matrix_1.2-14     rmarkdown_1.10   
## [22] labeling_0.3      stringr_1.3.1     munsell_0.5.0    
## [25] broom_0.5.0       compiler_3.5.1    janeaustenr_0.1.5
## [28] pkgconfig_2.0.2   htmltools_0.3.6   openssl_1.0.2    
## [31] tidyselect_0.2.5  tibble_1.4.2      gridExtra_2.3    
## [34] fansi_0.4.0       crayon_1.3.4      withr_2.1.2      
## [37] MASS_7.3-50       SnowballC_0.5.1   grid_3.5.1       
## [40] nlme_3.1-137      jsonlite_1.5      gtable_0.2.0     
## [43] magrittr_1.5      units_0.6-1       scales_1.0.0     
## [46] tokenizers_0.2.1  cli_1.0.1         stringi_1.2.4    
## [49] reshape2_1.4.3    farver_1.0        NLP_0.2-0        
## [52] xml2_1.2.0        tools_3.5.1       glue_1.3.0       
## [55] tweenr_1.0.0      purrr_0.2.5       parallel_3.5.1   
## [58] yaml_2.2.0        tm_0.7-6          colorspace_1.3-2 
## [61] bindr_0.1.1       modeltools_0.2-22</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
