<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Javier Herrero" />


<title>Twitter coverage of the Biology of Genomes 2019 conference</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lazappi's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Hashtags
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="bog2019.html">BoG19</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/jherrero/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Twitter coverage of the Biology of Genomes 2019 conference</h1>
<h4 class="author"><em>Javier Herrero</em></h4>
<h4 class="date"><em>2019-05-08 19:59:44</em></h4>

</div>


<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>An analysis of tweets from the Biology of Genomes 2019 conference. A total of 886 tweets from 307 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1">
<h1><span class="header-section-number">2</span> Timeline</h1>
<div id="tweets-by-day" class="section level2">
<h2><span class="header-section-number">2.1</span> Tweets by day</h2>
<p><img src="figures/bog2019/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2">
<h2><span class="header-section-number">2.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2019-05-07 - 2019-05-08 in the America/New_York timezone.</p>
<p><img src="figures/bog2019/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1">
<h1><span class="header-section-number">3</span> Users</h1>
<div id="top-tweeters" class="section level2">
<h2><span class="header-section-number">3.1</span> Top tweeters</h2>
<p><img src="figures/bog2019/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-original-tweeters" class="section level2">
<h2><span class="header-section-number">3.2</span> Top original tweeters</h2>
<p><img src="figures/bog2019/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-proportion" class="section level2">
<h2><span class="header-section-number">3.3</span> Retweet proportion</h2>
<p><img src="figures/bog2019/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-time" class="section level2">
<h2><span class="header-section-number">3.4</span> Top tweeters by time</h2>
<p><img src="figures/bog2019/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2">
<h2><span class="header-section-number">3.5</span> Top tweeters by day</h2>
<div id="all-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.1</span> All tweets</h3>
<div id="day-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/bog2019/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/bog2019/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.2</span> Original tweets</h3>
<div id="day-1-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="sources" class="section level2">
<h2><span class="header-section-number">3.6</span> Sources</h2>
<p><img src="figures/bog2019/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="networks" class="section level1">
<h1><span class="header-section-number">4</span> Networks</h1>
<div id="replies" class="section level2">
<h2><span class="header-section-number">4.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by page rank.</p>
<p><img src="figures/bog2019/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2">
<h2><span class="header-section-number">4.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Node colour, label size and alpha adjusted according to PageRank score.</p>
<p><img src="figures/bog2019/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets" class="section level1">
<h1><span class="header-section-number">5</span> Retweets</h1>
<div id="retweet-proportion-1" class="section level2">
<h2><span class="header-section-number">5.1</span> Retweet proportion</h2>
<p><img src="figures/bog2019/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-count" class="section level2">
<h2><span class="header-section-number">5.2</span> Retweet count</h2>
<p><img src="figures/bog2019/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-retweets" class="section level2">
<h2><span class="header-section-number">5.3</span> Top retweets</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
athmapai
</td>
<td style="text-align:left;">
Recent or soon to be grads: looking for a postdoc in gene regulation, RNA, and genomics? I’d love to chat! Come find me at #BoG19! I’ll be presenting poster #185 on Thursday <a href="https://t.co/FBsPTVH0Tn" class="uri">https://t.co/FBsPTVH0Tn</a>
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Transatlantic wifi and I can feel the genome tribes gathering for #bog19 - a little history for those not acquainted with this meeting
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Olga Sigalova, Nice to see discussion of Broad vs narrow promoters (once identified by CAGE back in Fantom3), whereas broad promoters tend to be housekeeping and narrow drive expression of more specific genes; reviewed in: <a href="https://t.co/0IQWlNnKMn" class="uri">https://t.co/0IQWlNnKMn</a> #bog19
</td>
<td style="text-align:right;">
11
</td>
</tr>
<tr>
<td style="text-align:left;">
aaronquinlan
</td>
<td style="text-align:left;">
Really fun talk from <span class="citation">@BeEngelhardt</span> on a Bayesian approach for optimizing experimental design to maximize cell type discovery from single-cell sequencing experiments. Calling it GT-TS (Good-Toulmin like estimator via Thompson sampling). Paper here: <a href="https://t.co/uWm7ySP4lG" class="uri">https://t.co/uWm7ySP4lG</a> #bog19
</td>
<td style="text-align:right;">
10
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Now Mike Schatz <span class="citation">@mike_schatz</span> on assembling many tomato 🍅 genomes 🧬 and structural variants (fellow <span class="citation">@GenomicsAtJHU</span> member)</p>
<p>Uses <span class="citation">@nanopore</span> and HiC data, then RaGOO software</p>
<p>Code 💻: <a href="https://t.co/ljO0NzR0FB" class="uri">https://t.co/ljO0NzR0FB</a> 📜: <a href="https://t.co/ge0PJLQCix" class="uri">https://t.co/ge0PJLQCix</a> 🖇️: <a href="https://t.co/QAmuwXDF4E" class="uri">https://t.co/QAmuwXDF4E</a></p>
#BoG19
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
PLOSBiology
</td>
<td style="text-align:left;">
If you’ve been “Scooped,” remember <span class="citation">@PLOSBiology</span>’s anti-scooping policy; come to us within 6 months of “scooping” and we’ll consider it regardless. Read our policy here: <a href="https://t.co/hMd2vrXPqU" class="uri">https://t.co/hMd2vrXPqU</a> or if you’re at #BoG19 you can just talk to <span class="citation">@RoliRoberts</span>… <a href="https://t.co/leZMMON2rh" class="uri">https://t.co/leZMMON2rh</a>
</td>
<td style="text-align:right;">
9
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Arnav Moudgil: single cell RNA-seq AND Transcription factor binding. This is the first approach to map transcription factor; details here: <a href="https://t.co/i34FBRTzQH" class="uri">https://t.co/i34FBRTzQH</a> #bog19
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Just in time for #bog19, <span class="citation">@genomeresearch</span> posts early access version (today) of our paper where we found 2250 bacterial genomes that had human contamination, plus 3,437 “proteins” that are erroneous translations of these human contaminants <a href="https://t.co/koQRkmLRPw" class="uri">https://t.co/koQRkmLRPw</a>
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:left;">
DanielDSeaton
</td>
<td style="text-align:left;">
<p>We have a new preprint! Single-cell RNA-seq of differentiating iPS cells reveals dynamic genetic effects on gene expression. <a href="https://t.co/aJDcNZRfkT" class="uri">https://t.co/aJDcNZRfkT</a></p>
<p>Work being presented this afternoon at #BoG19 by <span class="citation">@AnnaCuomo92</span>, Poster 68.</p>
1/ <a href="https://t.co/ub93uZ7Dae" class="uri">https://t.co/ub93uZ7Dae</a>
</td>
<td style="text-align:right;">
8
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="favourites" class="section level1">
<h1><span class="header-section-number">6</span> Favourites</h1>
<div id="favourite-proportion" class="section level2">
<h2><span class="header-section-number">6.1</span> Favourite proportion</h2>
<p><img src="figures/bog2019/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="favourite-count" class="section level2">
<h2><span class="header-section-number">6.2</span> Favourite count</h2>
<p><img src="figures/bog2019/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-favourites" class="section level2">
<h2><span class="header-section-number">6.3</span> Top favourites</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
183
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Love it that the genome scaffolders used by <span class="citation">@mike_schatz</span> for the tomato genome are called SALSA2 and RaGOO. Coincidence? #BoG19
</td>
<td style="text-align:right;">
49
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
#bog19 the first BOG without the portrait of Watson. Finally. <a href="https://t.co/i42jfyMKdt" class="uri">https://t.co/i42jfyMKdt</a>
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
lukejostins
</td>
<td style="text-align:left;">
At Biology of Genomes this week. A bit jet lagged for live tweeting right now, but jesus there are some unsettlingly gifted graduate students giving talks tonight #BoG19
</td>
<td style="text-align:right;">
43
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Transatlantic wifi and I can feel the genome tribes gathering for #bog19 - a little history for those not acquainted with this meeting
</td>
<td style="text-align:right;">
39
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Congratulations Pavel Pevzner for winning the ACM Kanellakis Award! <a href="https://t.co/ebOw1uH4Rn" class="uri">https://t.co/ebOw1uH4Rn</a> I just learned this in the midst of the computational biology session at #Bog19, how appropriate
</td>
<td style="text-align:right;">
34
</td>
</tr>
<tr>
<td style="text-align:left;">
vagar112
</td>
<td style="text-align:left;">
Looking forward to Biology of Genomes 2019 #BoG19 Please drop by my poster, would love to chat with anyone interested! For those who cannot make it to the conference, here is a free copy of the poster 🙂 <a href="https://t.co/yvY6IS5PIv" class="uri">https://t.co/yvY6IS5PIv</a>
</td>
<td style="text-align:right;">
34
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Olga Sigalova, Nice to see discussion of Broad vs narrow promoters (once identified by CAGE back in Fantom3), whereas broad promoters tend to be housekeeping and narrow drive expression of more specific genes; reviewed in: <a href="https://t.co/0IQWlNnKMn" class="uri">https://t.co/0IQWlNnKMn</a> #bog19
</td>
<td style="text-align:right;">
33
</td>
</tr>
<tr>
<td style="text-align:left;">
aaronquinlan
</td>
<td style="text-align:left;">
Really fun talk from <span class="citation">@BeEngelhardt</span> on a Bayesian approach for optimizing experimental design to maximize cell type discovery from single-cell sequencing experiments. Calling it GT-TS (Good-Toulmin like estimator via Thompson sampling). Paper here: <a href="https://t.co/uWm7ySP4lG" class="uri">https://t.co/uWm7ySP4lG</a> #bog19
</td>
<td style="text-align:right;">
32
</td>
</tr>
<tr>
<td style="text-align:left;">
obahcall
</td>
<td style="text-align:left;">
Excited for #BoG19! This will be my 15th consecutive BoG and always one of my favorites. Every year the organizers &amp;amp; <span class="citation">@cshlmeetings</span> manage to represent the best across a broad range of genomics &amp;amp; to host a meeting that fosters collaborations that have been central to genomics.
</td>
<td style="text-align:right;">
29
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="quotes" class="section level1">
<h1><span class="header-section-number">7</span> Quotes</h1>
<div id="quote-proportion" class="section level2">
<h2><span class="header-section-number">7.1</span> Quote proportion</h2>
<p><img src="figures/bog2019/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="quote-count" class="section level2">
<h2><span class="header-section-number">7.2</span> Quote count</h2>
<p><img src="figures/bog2019/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-quotes" class="section level2">
<h2><span class="header-section-number">7.3</span> Top quotes</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Awesomics
</td>
<td style="text-align:left;">
The Biology of Genomes (#BoG19) now underway <a href="https://t.co/uJpygePI9e" class="uri">https://t.co/uJpygePI9e</a>. Access live streamed talks, abstract PDF, and video archive (for 6 weeks) at <a href="https://t.co/L8JiHY0BFH" class="uri">https://t.co/L8JiHY0BFH</a> and enter “99” in the ‘Payment Authorized’ field. Cost=$99. Thanks <span class="citation">@cshlmeetings</span>! <a href="https://t.co/a4zUxtCduV" class="uri">https://t.co/a4zUxtCduV</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
AnnaCuomo92
</td>
<td style="text-align:left;">
Great Illustration and great meeting so far! #BoG19 <a href="https://t.co/MbRJgVUS5E" class="uri">https://t.co/MbRJgVUS5E</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="media" class="section level1">
<h1><span class="header-section-number">8</span> Media</h1>
<div id="media-count" class="section level2">
<h2><span class="header-section-number">8.1</span> Media count</h2>
<p><img src="figures/bog2019/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-media" class="section level2">
<h2><span class="header-section-number">8.2</span> Top media</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
183
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
#bog19 the first BOG without the portrait of Watson. Finally. <a href="https://t.co/i42jfyMKdt" class="uri">https://t.co/i42jfyMKdt</a>
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
Taking a detour to the Brooklyn Botanic Garden on the way to #bog19. I’m so excited to be back!! <a href="https://t.co/CiObVNKySH" class="uri">https://t.co/CiObVNKySH</a>
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:left;">
ChrisOmics
</td>
<td style="text-align:left;">
My girl <span class="citation">@ruthie_johnson</span> talking about her new method BEAVR estimating proportions of causal SNPs in a region at #BoG19. <a href="https://t.co/KwbHQerO87" class="uri">https://t.co/KwbHQerO87</a>
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
PLOSBiology
</td>
<td style="text-align:left;">
If you’ve been “Scooped,” remember <span class="citation">@PLOSBiology</span>’s anti-scooping policy; come to us within 6 months of “scooping” and we’ll consider it regardless. Read our policy here: <a href="https://t.co/hMd2vrXPqU" class="uri">https://t.co/hMd2vrXPqU</a> or if you’re at #BoG19 you can just talk to <span class="citation">@RoliRoberts</span>… <a href="https://t.co/leZMMON2rh" class="uri">https://t.co/leZMMON2rh</a>
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:left;">
JedMSP
</td>
<td style="text-align:left;">
<p>Come see me at poster 46 to talk about this hell site with me</p>
#BoG19 <a href="https://t.co/5PQZtqyFzx" class="uri">https://t.co/5PQZtqyFzx</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
wbmei
</td>
<td style="text-align:left;">
Arrive the beautiful cold spring harbor campus and exited about learning more genetics and genomics in the next couple days #BoG19 <a href="https://t.co/ak2apnJHXD" class="uri">https://t.co/ak2apnJHXD</a>
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
bpasaniuc
</td>
<td style="text-align:left;">
Great job from <span class="citation">@ruthie_johnson</span> presenting the regional polygenic estimation work at the #BoG19 ! <a href="https://t.co/WTwsxEDLdQ" class="uri">https://t.co/WTwsxEDLdQ</a>
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
mike_schatz
</td>
<td style="text-align:left;">
Opening talk for #bog19 by <span class="citation">@arjunrajlab</span>. Non-genetic variability and therapy resistance in cancer. Should be a great meeting! <a href="https://t.co/hWlOtiH5yH" class="uri">https://t.co/hWlOtiH5yH</a>
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:left;">
rachelCSHL
</td>
<td style="text-align:left;">
Welcome to the Biology of Genomes conference #bog19 <a href="https://t.co/D2PBCMX5zV" class="uri">https://t.co/D2PBCMX5zV</a>
</td>
<td style="text-align:right;">
18
</td>
</tr>
</tbody>
</table>
<div id="most-liked-media-image" class="section level3">
<h3><span class="header-section-number">8.2.1</span> Most liked media image</h3>
<div class="figure">
<img src="http://pbs.twimg.com/media/D5-M0UeU8AAd3BC.jpg" />

</div>
</div>
</div>
</div>
<div id="tweet-text" class="section level1">
<h1><span class="header-section-number">9</span> Tweet text</h1>
<div id="word-cloud" class="section level2">
<h2><span class="header-section-number">9.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/bog2019/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2">
<h2><span class="header-section-number">9.2</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 5 times.</p>
<p><img src="figures/bog2019/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2">
<h2><span class="header-section-number">9.3</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/bog2019/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset">
<h3><span class="header-section-number">9.3.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered">
<h4>Topic 1</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>SMAC-seq by Georgi Marinov and team re-captures the peaks by DNA-seq peak but also overlaps H4S37C chemical nucleosome mapping peaks</p>
<p>Their profiles are nicely phased with Cse4 ChIP-seq as well</p>
<p>(I don’t know what Cse4 nor H4S37C do! Tweet and later revisit? :P)</p>
#bog19
</td>
<td style="text-align:right;">
0.9954054
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Hannah Currant, on UK Biobank, 500K individuals, 96M variants: focus on retinal phenotype. Retina inspected for physical differences (example: thickness, as “endophenotype”), which is connected with diseases/biology. ~31K GWAS- identify loci associated with retina problems #bog19
</td>
<td style="text-align:right;">
0.9952227
</td>
</tr>
<tr>
<td style="text-align:left;">
aaronquinlan
</td>
<td style="text-align:left;">
Really fun talk from <span class="citation">@BeEngelhardt</span> on a Bayesian approach for optimizing experimental design to maximize cell type discovery from single-cell sequencing experiments. Calling it GT-TS (Good-Toulmin like estimator via Thompson sampling). Paper here: <a href="https://t.co/uWm7ySP4lG" class="uri">https://t.co/uWm7ySP4lG</a> #bog19
</td>
<td style="text-align:right;">
0.9952227
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Andrew Jaffe (partnering with companies in BrainSeq consortium). In this talk focus on hippocampus, laser capture, bulk RNAseq from LCM of dentate gyrus (DG). Then DG specific eQTL, finding eQTLs associated with schizophrenia. #bog19
</td>
<td style="text-align:right;">
0.9948098
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Next Barbara Engelhardt <span class="citation">@BeEngelhardt</span> on discovering &amp;amp; distinguishing cell types, as part of a larger goal of producing cell maps</p>
<p>Work by now PhD-alumni Bianca Dumitrascu <span class="citation">@datumbeet</span></p>
Being #bayesian yields incredible rewards <a href="https://t.co/LhSBNAw4Kg" class="uri">https://t.co/LhSBNAw4Kg</a> <a href="https://t.co/WUaCFnM1sq" class="uri">https://t.co/WUaCFnM1sq</a> #BoG19
</td>
<td style="text-align:right;">
0.9948098
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
HC: Clusters SNPs in a 24-dimensional space from the OCT data, and fall out into 3 quite separate clusters. One of the clusters with MR support a GCIPL &amp;lt;=&amp;gt; Glaucoma link (which is consistent with epidemiology / ophthalmology) #bog19. She handles the questions well
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
GM — SMAC single molecule accessibility of the chromatin. Reproduces DNase-seq + intra-nucleosome space. Single molecule resolution show differences in the population with 10-20% (?) cells with high accessibility signal, relates to transcription #BoG19
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
abarysh
</td>
<td style="text-align:left;">
Just posted an ad for a compbio position in my group (+ other Calico opportunities) on the board next to CSHL meetings office (Grace Auditorium, downstairs). #BOG19 folks on the job market: check it out and/or reach out to me for a chat–happy to give all the details in person!
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
obahcall
</td>
<td style="text-align:left;">
So glad to see #BoG19 continues to lead in the genome way. (Nearly) all talks have been tweetable. And most speakers encourage: showing twitter handle, relevant preprints, software &amp;amp; QR code. Discussion in room &amp;amp; online goes much deeper from this- esp when we can read preprint.
</td>
<td style="text-align:right;">
0.9943189
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Next up, Andrew Jaffe <span class="citation">@andrewejaffe</span> from <span class="citation">@LieberInstitute</span> on “Cell type-specific genetic regulation of expression in the granule cell layer of the human dentate gyrus” with funding from <span class="citation">@AstellasUS</span></p>
<p><a href="https://t.co/Z7lmKHTej8" class="uri">https://t.co/Z7lmKHTej8</a></p>
<p>Related to <a href="https://t.co/ptjNrpAFGJ" class="uri">https://t.co/ptjNrpAFGJ</a> from #bog18</p>
#bog19
</td>
<td style="text-align:right;">
0.9940369
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered">
<h4>Topic 2</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
RJ: Causal effect size distribution parameterized by effect size (beta) and number of causal SNPs (k) (Zhang et al. NG 2018). Want to know how is heritability divided - centered on 1 SNP or distributed among multiple SNPs, has implication to effect distr at regional level. #BoG19
</td>
<td style="text-align:right;">
0.9954054
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
MS: greedily identify the most diverse species from short-read data to identify candidates for long-read sequencing. PromethION improved sequencing throughput dramatically. Read length ~10-100kb length, rarily &amp;gt; 1 Mb. #BoG19
</td>
<td style="text-align:right;">
0.9950248
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
RJ: BEAVR estimates local polygenicity from GWAS summary statistics. Use GWAS “noisy” effect size estimates to infer underlying effect size distr. Can estimate # of causal SNPs across varying level of heritability. #BoG19
</td>
<td style="text-align:right;">
0.9950248
</td>
</tr>
<tr>
<td style="text-align:left;">
nanopore
</td>
<td style="text-align:left;">
Will you be at Biology of Genomes next week? Register for our free lunch &amp;amp; seminar on Thursday 9th May. Steven Salzberg is giving a talk on ‘The Redwood Genome Project: sequencing and assembling the largest trees in the world’ join us here: <a href="https://t.co/gq8U4SXRh8" class="uri">https://t.co/gq8U4SXRh8</a> #bog19 <a href="https://t.co/GTbHxxK06n" class="uri">https://t.co/GTbHxxK06n</a>
</td>
<td style="text-align:right;">
0.9950248
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Related approach: allelic heterogeneity (&amp;gt; 1 causal SNP in a locus)</p>
<p><span class="citation">@zarlab</span> says it’s detectable using a simple ratio formula (that I’m not setup to type, sry <span class="citation">@michaelhoffman</span> :P)</p>
Anyhow, now that there are many fine mapping methods, we can 👀 at locus association tests #BoG19
</td>
<td style="text-align:right;">
0.9948098
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
MS: Good final chromosome level leveraging reference and HiC for validation, polishing with short reads using FreeBayes as the polisher. Shows an example of the “easy to break free” tomato (jointless) and “long stem” tomato which has a negative epistatic interaction #BoG19
</td>
<td style="text-align:right;">
0.9948098
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
EE. Caviar works on a classic Bayesian scheme (LLR) on a set of causal SNPs - which also fits this Allelic heterogeneity problems - similar LLR test for &amp;gt;1 causal SNP. Finds a lot of (local) allelic heterogeneity #bog19
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Michael Shatz, 100 genomes in 100 days. Tomato genomes, 15K varieties. Unappreciated that tomatos is broadly used for GWAS and traits. Also full of structural variations, but difficult to capture: here addressed by long reads in a SV consortium. #bog19
</td>
<td style="text-align:right;">
0.9943189
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
EE: (2) is fine-mapping, and fine-mapping framework can be extended to detect allelic heterogeneity (locus having &amp;gt; 1 causal variants). eQTL and GWAS datasets show prevalent allelie heterogeneity. #BoG19
</td>
<td style="text-align:right;">
0.9940369
</td>
</tr>
<tr>
<td style="text-align:left;">
sorrywm
</td>
<td style="text-align:left;">
.<span class="citation">@ruthie_johnson</span>’s BEAVR method chows down on the tough wood of estimating how many causal SNPs in a region contribute to that region’s heritability for a complex trait, using GWAS summary stats and LD as input. #BoG19
</td>
<td style="text-align:right;">
0.9940369
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-3" class="section level4 unnumbered">
<h4>Topic 3</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Coronary artery disease is the leading global cause of mortality</p>
<p>Current risk assessment of heart attack reviewed by Khera et al <a href="https://t.co/7BJCbpN2xv" class="uri">https://t.co/7BJCbpN2xv</a> based on non-🧬metrics</p>
but <span class="citation">@skathire</span> has now shown that polygenic risk score is the first risk factor for this disease #BoG19 <a href="https://t.co/UuHPODF7cq" class="uri">https://t.co/UuHPODF7cq</a>
</td>
<td style="text-align:right;">
0.9952227
</td>
</tr>
<tr>
<td style="text-align:left;">
brimittleman
</td>
<td style="text-align:left;">
.<span class="citation">@skathire</span> polygenic risk score for coronary heart disease trained on 6.6 million variants. Top 1% in score have 5 fold increased risk. Similar odds ratio to inscreased risk from monogenic model #bog19
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
ficalvert
</td>
<td style="text-align:left;">
Thought I’d beaten jet lag by forcing myself to stay up late on my first night in New York. My 5am wake up this morning was a brutal reminder that you can never beat jet lag! Excited to go explore the city after using my early start to work on my #BoG19 slides!
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
Ready for #BoG19. Poster tube to present the work of Cecilia (bright BSc student) and Garima (bright postdoc) on using vg for somatic variant detection. All started while chatting to <span class="citation">@erikgarrison</span> last year at #BoG18. At the airport, free coffee refills. Will need them! 😎
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Sek takes us through the construction of polygenic risk score - from MI GWAS, selected best score from 20 with first round of UK BioBank, then test on remainder. PRS is not correlated to the ACC/AHA risk score #BoG19
</td>
<td style="text-align:right;">
0.9943189
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
SK: extend PRS to diseases beyond MI: atrial fibrillation, diabetes, IBD, breast cancer. Anywhere between 2-6% of population has &amp;gt; 3x risk. For weight/BMI, decile of PRS could differ in 30 lb in weight. #BoG19
</td>
<td style="text-align:right;">
0.9943189
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
in Q&amp;amp;A, added that of the top 20 or so blood protein differences, 15 have some literature / animal model evidence that they are related to Alzheimer. (The AD score includes ApoE effect, so not necessarily surprising to see difference in ApoE) #BoG19
</td>
<td style="text-align:right;">
0.9940369
</td>
</tr>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
Morning session seems to be opt-in for twitter again, despite the guidelines published in the abstract book. Session chairs being over-cautious? Also, talks have been culled from 15’+5’ to 12’+3’. Why? From 9 till 12 we get 3 hours, enough time for 8 x 20min + 20min break #BoG19
</td>
<td style="text-align:right;">
0.9937254
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
SK: polygenic score in 300k individuals was a nice bell curve. The score has r = 0.02 with the ACC/AHA risk score. Suggests PRS is orthogonal to what is used in clinical practice. #BoG19
</td>
<td style="text-align:right;">
0.9937254
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Took me a bit to find the actual figure I wanted to tweet initially (it was under “box 1” instead of figures)</p>
<p>This figure shows what the current non-🧬 (DNA) metrics that are used to assess risk for heart attacks (coronary artery disease)</p>
#BoG19 <span class="citation">@skathire</span> <a href="https://t.co/85BnDvJY1U" class="uri">https://t.co/85BnDvJY1U</a>
</td>
<td style="text-align:right;">
0.9937254
</td>
</tr>
<tr>
<td style="text-align:left;">
ShobanaSekr
</td>
<td style="text-align:left;">
Super excited to attend the biology of genomes conference #BoG19 @ cold spring harbor for the first time and with my new lab! Looking forward to the awesome line up of talks &amp;amp; posters and of course the beautiful weather and campus!
</td>
<td style="text-align:right;">
0.9937254
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered">
<h4>Topic 4</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
.<span class="citation">@nonfiniteloop</span> — Can we detect post zygotic variations. Re-purpose GTEx data. Develop method to detect mutations from bulk RNA-seq and validate in Sun exposed/unexposed skin. More mutations in UV exposed skin, increase in UV-induced mut. signature #BoG19
</td>
<td style="text-align:right;">
0.9958784
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Rockweiler/2: measuring skin exposed/not to UVs showing more skin mutation in caucasians if compared to African American donors. identify 79 mutations/donor, 4/tissues, 7% overlap, &amp;gt;82-88% validation rate. Mutation rate very variable across tissues and samples. #bog19
</td>
<td style="text-align:right;">
0.9952227
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Rockweiler/3. Mutations are tissue specific (example: C-&amp;gt;T in skin). Brain less mutations. Age/timing? Shared mutation is more ancient -&amp;gt; Surprise: 80% of mutations happen between zygote and gastrula! (surprise: in so few cell cycles!) Do not miss cell specific ones? #bog19
</td>
<td style="text-align:right;">
0.9952227
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Nicole Rockweiler, whole body maps postzygotic mutations in humans (= after fertilization through lifespan). The genome is not the same in all cells or in twins! Discuss: where, when? Repurposing GTEx data, 17K RNA-seq libraries, reconstruction of genome data from RNA seq. #bog19
</td>
<td style="text-align:right;">
0.9952227
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
NW - This is a relatively new field. 3 questions - can we detect these mutations. Repurposing GTex 964 postmortem donors for post zygotic mutation. Use sun exposed vs non sun exposed skin - see increase in mutation burden in sun exposed skin in caucasians #bog19
</td>
<td style="text-align:right;">
0.9950248
</td>
</tr>
<tr>
<td style="text-align:left;">
obahcall
</td>
<td style="text-align:left;">
Appreciation for our wonderful #BoG19 organizers, who are also illustrious scientists celebrating recent well deserved recognitions!<br />
Matt Hurles <span class="citation">@mehurles</span> elected FRS <span class="citation">@royalsociety</span> Elaine Ostrander elected <span class="citation">@theNASciences</span> Jay Shendure <span class="citation">@JShendure</span> Lounsbery award <span class="citation">@theNASciences</span>
</td>
<td style="text-align:right;">
0.9950248
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>NB <span class="citation">@nonfiniteloop</span> found the lowest mutation burden in the brain, and highest in liver followed by EBV (missed info here) &amp;amp; esophagus</p>
<p>NB’s main questions: Can we detect postzygotic mutations? where are they? &amp;amp; when they occur?</p>
<p>On Q3: some tissues increase 🧬mut with age</p>
#BoG19
</td>
<td style="text-align:right;">
0.9950248
</td>
</tr>
<tr>
<td style="text-align:left;">
NeBanovich
</td>
<td style="text-align:left;">
NR: Developed method to identify postzygotic mutations from bulk RNA-seq data. Showing results validating known biology - Caucasian donors w/increase mutations in sun exposed skin vs non-sun exposed skin. #bog19
</td>
<td style="text-align:right;">
0.9948098
</td>
</tr>
<tr>
<td style="text-align:left;">
skathire
</td>
<td style="text-align:left;">
<p>Really beautiful talk by <span class="citation">@nonfiniteloop</span> on post-zygotic mutations in GTeX</p>
<p>What looks to be first survey of somatic mutations across many tissues</p>
<p>Tissue with highest rate - liver Lowest rate - brain</p>
<p>Rate in most tissues is age dependent but not all</p>
<p>Cc: <span class="citation">@jaiswalmdphd</span></p>
#BoG19
</td>
<td style="text-align:right;">
0.9948098
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
NR: Can we detect? When and where did post-zyg mutations occur? Use GTex data (~960 normal donors, ~17 tissues per donor, 17K RNA-seq libraries). Main question how to go from RNAseq reads to post-zyg mutations? #BoG19
</td>
<td style="text-align:right;">
0.9948098
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered">
<h4>Topic 5</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Daniel <span class="citation">@vervacity</span> clustered ATAC peaks to find dynamic accessible regions, then looked into H3K27ac and H3K4me1 data</p>
<p>Lead to the question: how does DNA sequence lead to these molecular phenotypes? (the dynamic regions)</p>
<p>Goal: use DNA sequence to predict chromatin activ.</p>
#bog19
</td>
<td style="text-align:right;">
0.9955747
</td>
</tr>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
OS — broad promoters &amp;lt;=&amp;gt; housekeeping genes ; narrow promoters and low variation =&amp;gt; GO: developments, regulation of transcription; narrow promoters and high variation =&amp;gt; Metabolic processes (lots of TAT box as well) #BoG19
</td>
<td style="text-align:right;">
0.9950248
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Daniel Kim, regulatory DNA sequences and networks in keratinocytes differentiation, 6 days; looking at accessibility (ATAC-seq) and expression (high density), and few more histone modifications. Neural networks used to map accessibility, pre-trained models are accurate #bog19.
</td>
<td style="text-align:right;">
0.9950248
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Arjun Raj: drug resistance in a dish: melanoma cells that are drug resistant even without previous selection because they are in a certain “state”. Phenotype &amp;lt;-&amp;gt; molecular state? Lentiviral barcoding to follow the “twin” resistant cells. Then genomics characterized. #bog19
</td>
<td style="text-align:right;">
0.9948098
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
DK - first mention of deep neural networks. Does some nice transfer learning from epigenome/ENCODE deep neural networks. Pretty good model regression from DNA to activity. (Deep Neural Networks for the win!) #bog19
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Saumya Agrawal clustered the HiC regions, then found the genes in those clusters and identified enriched gene ontologies in these gene clusters</p>
She also noted before that the HiC enrichment hadn’t been shown for lncRNA before (I missed some of the context sry!) #BoG19 <span class="citation">@carninci</span>
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Olga Sigalova, Nice to see discussion of Broad vs narrow promoters (once identified by CAGE back in Fantom3), whereas broad promoters tend to be housekeeping and narrow drive expression of more specific genes; reviewed in: <a href="https://t.co/0IQWlNnKMn" class="uri">https://t.co/0IQWlNnKMn</a> #bog19
</td>
<td style="text-align:right;">
0.9943189
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Daneil’s model <span class="citation">@vervacity</span> finds something like motifs without TF binding data (they are supported by TF ChIP-seq data)</p>
His image reminded me of LASSO since the contribution of most base pairs goes to 0, and only a few have signal left (which is what looks like a motif) #bog19
</td>
<td style="text-align:right;">
0.9943189
</td>
</tr>
<tr>
<td style="text-align:left;">
preetiaahir
</td>
<td style="text-align:left;">
Gene regulation insight in development with promoter complexicity. Olga Sigalova from #EMBL talks about developing a model to predict the expression variation in promoter genes #RandomForest #Machinelearning #GeneRegulation #Development #BoG19
</td>
<td style="text-align:right;">
0.9943189
</td>
</tr>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
Daniel S. Kim — In sillico mutagenesis to infer synergestic interactions btw motifs within a locus. Build a pairwise full matrix of synergies in skin differentiation to define a “grammar” of motifs (only pairs of motifs) #BoG19
</td>
<td style="text-align:right;">
0.9943189
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered">
<h4>Topic 6</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
cshlmeetings
</td>
<td style="text-align:left;">
Unable to make it to next week’s #BoG19? Live stream it! Each $99 subscription includes live stream of talks, PDF of the abstract book, and six-week access to #BoG19’s video archive. Register online: <a href="https://t.co/fMcgEzbvys" class="uri">https://t.co/fMcgEzbvys</a> and enter “99” in the ‘Payment Authorized’ field. <a href="https://t.co/AHcQxt1Sg6" class="uri">https://t.co/AHcQxt1Sg6</a>
</td>
<td style="text-align:right;">
0.9950248
</td>
</tr>
<tr>
<td style="text-align:left;">
saketkc
</td>
<td style="text-align:left;">
My poster with <span class="citation">@WenzhengLi</span> and Andrew Smith for #bog19! We developed a simple method for finding actively-translating ORFs from Ribo-seq data. Performs well at both short and long ORFs. Code: <a href="https://t.co/s9Z77yGD2T" class="uri">https://t.co/s9Z77yGD2T</a> (also on pip and bioconda) Poster: <a href="https://t.co/rKwfLQQVbI" class="uri">https://t.co/rKwfLQQVbI</a>. <a href="https://t.co/gXdXAPqpEC" class="uri">https://t.co/gXdXAPqpEC</a>
</td>
<td style="text-align:right;">
0.9950248
</td>
</tr>
<tr>
<td style="text-align:left;">
sorrywm
</td>
<td style="text-align:left;">
Interesting finding from Yi Qiao’s talk: “mis-classified” sub-clones from breast cancer longitudinal dataset using scBayes (actually pre-treatment but classified as post-treatment) show that doxorubicin treatment produces selection on standing variation #BoG19
</td>
<td style="text-align:right;">
0.9948098
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
AR does a lentiviral infection followed by splitting, keeping one set of cells frozen. This then allows one to take one set of clones into a cellular phenotyping assay (eg, resistance assay) and then take bar codes back the frozen cells - they call this: “time machine” #bog19
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
NeBanovich
</td>
<td style="text-align:left;">
YQ: Missing link is tying genomic subclocal structure to transcriptional structure - can now use scRNA-seq and bulk RNAseq to tie these data together. However not trivial, largely due to low coverage in scRNA-seq data #bog19
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Final speaker for this #BoG19 session: Yi Qiao from the Marth Lab <span class="citation">@MarthGabor</span> on studying heterogeneity in cancer somatic subclones</p>
<p>Earlier methods: PyClone, SciClone, SubcloneSeeker</p>
<p>Presenting today is a “Dream come true” ^^</p>
<a href="https://t.co/tb1tqYpVet" class="uri">https://t.co/tb1tqYpVet</a> <a href="https://t.co/067j2RdgLU" class="uri">https://t.co/067j2RdgLU</a>
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>BE’s work <span class="citation">@BeEngelhardt</span> depends on accurate cell markers</p>
<p>They compare against Seurat et al, bioRxiv, 2018 <a href="https://t.co/Lx1bDbyLhw" class="uri">https://t.co/Lx1bDbyLhw</a></p>
<p>Hierarchy is missed in some previous methods. Addressed in scGeneFit (fast: 2 sec)</p>
Code: <a href="https://t.co/WzQUpaam52" class="uri">https://t.co/WzQUpaam52</a> 📜: <a href="https://t.co/LhSBNAw4Kg" class="uri">https://t.co/LhSBNAw4Kg</a> #BoG19
</td>
<td style="text-align:right;">
0.9945755
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>YQ (<span class="citation">@preetiaahir</span> <span class="citation">@MarthGabor</span>) applied scBayes to scDNAseq and scRNAseq data on longitudinal breast cancer (some treated with doxorubicin: I don’t know what this does 😅)</p>
<p>Google <a href="https://t.co/44BLpBgUOR" class="uri">https://t.co/44BLpBgUOR</a>: Doxorubicin … slows or stops the growth of cancer cells by blocking</p>
#BoG19
</td>
<td style="text-align:right;">
0.9943189
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>BE <span class="citation">@BeEngelhardt</span> &amp;amp; <span class="citation">@datumbeet</span> developed GT-TS and used it with simulated mouse cell atlas data</p>
<p>New results (not on pre-print) show much closer results to Oracle (see Fig 2 C &amp;amp; D for prior results)</p>
<p><a href="https://t.co/jePa2B7plw" class="uri">https://t.co/jePa2B7plw</a></p>
#BoG19 <a href="https://t.co/IA8F6SPYf6" class="uri">https://t.co/IA8F6SPYf6</a>
</td>
<td style="text-align:right;">
0.9943189
</td>
</tr>
<tr>
<td style="text-align:left;">
cshlmeetings
</td>
<td style="text-align:left;">
It is indeed a great alternative for those unable to make it here in person. In addition to the live streamed talks are a PDF of the abstract book and 6-week access to #BoG19’s video archive. Register at <a href="https://t.co/fMcgEzbvys" class="uri">https://t.co/fMcgEzbvys</a> and enter “99” in the ‘Payment Authorized’ field. <a href="https://t.co/gZRGpanOOt" class="uri">https://t.co/gZRGpanOOt</a>
</td>
<td style="text-align:right;">
0.9943189
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1>Session info</h1>
<pre><code>## function (package = NULL) 
## {
##     z &lt;- list()
##     z$R.version &lt;- R.Version()
##     z$platform &lt;- z$R.version$platform
##     if (nzchar(.Platform$r_arch)) 
##         z$platform &lt;- paste(z$platform, .Platform$r_arch, sep = &quot;/&quot;)
##     z$platform &lt;- paste0(z$platform, &quot; (&quot;, 8 * .Machine$sizeof.pointer, 
##         &quot;-bit)&quot;)
##     z$locale &lt;- Sys.getlocale()
##     if (.Platform$OS.type == &quot;windows&quot;) {
##         z$running &lt;- win.version()
##     }
##     else if (nzchar(Sys.which(&quot;uname&quot;))) {
##         uname &lt;- system(&quot;uname -a&quot;, intern = TRUE)
##         os &lt;- sub(&quot; .*&quot;, &quot;&quot;, uname)
##         z$running &lt;- switch(os, Linux = if (file.exists(&quot;/etc/os-release&quot;)) {
##             tmp &lt;- readLines(&quot;/etc/os-release&quot;)
##             t2 &lt;- if (any(startsWith(tmp, &quot;PRETTY_NAME=&quot;))) sub(&quot;^PRETTY_NAME=&quot;, 
##                 &quot;&quot;, grep(&quot;^PRETTY_NAME=&quot;, tmp, value = TRUE)[1L]) else if (any(startsWith(tmp, 
##                 &quot;NAME&quot;))) sub(&quot;^NAME=&quot;, &quot;&quot;, grep(&quot;^NAME=&quot;, tmp, 
##                 value = TRUE)[1L]) else &quot;Linux (unknown distro)&quot;
##             sub(&quot;\&quot;(.*)\&quot;&quot;, &quot;\\1&quot;, t2)
##         } else if (file.exists(&quot;/etc/system-release&quot;)) {
##             readLines(&quot;/etc/system-release&quot;)
##         }, Darwin = {
##             ver &lt;- readLines(&quot;/System/Library/CoreServices/SystemVersion.plist&quot;)
##             ind &lt;- grep(&quot;ProductUserVisibleVersion&quot;, ver)
##             ver &lt;- ver[ind + 1L]
##             ver &lt;- sub(&quot;.*&lt;string&gt;&quot;, &quot;&quot;, ver)
##             ver &lt;- sub(&quot;&lt;/string&gt;$&quot;, &quot;&quot;, ver)
##             ver1 &lt;- strsplit(ver, &quot;.&quot;, fixed = TRUE)[[1L]][2L]
##             sprintf(&quot;%s %s %s&quot;, ifelse(as.numeric(ver1) &lt; 12, 
##                 &quot;OS X&quot;, &quot;macOS&quot;), switch(ver1, `6` = &quot;Snow Leopard&quot;, 
##                 `7` = &quot;Lion&quot;, `8` = &quot;Mountain Lion&quot;, `9` = &quot;Mavericks&quot;, 
##                 `10` = &quot;Yosemite&quot;, `11` = &quot;El Capitan&quot;, `12` = &quot;Sierra&quot;, 
##                 `13` = &quot;High Sierra&quot;, &quot;&quot;), ver)
##         }, SunOS = {
##             ver &lt;- system(&quot;uname -r&quot;, intern = TRUE)
##             paste(&quot;Solaris&quot;, strsplit(ver, &quot;.&quot;, fixed = TRUE)[[1L]][2L])
##         }, uname)
##     }
##     if (is.null(package)) {
##         package &lt;- grep(&quot;^package:&quot;, search(), value = TRUE)
##         keep &lt;- vapply(package, function(x) x == &quot;package:base&quot; || 
##             !is.null(attr(as.environment(x), &quot;path&quot;)), NA)
##         package &lt;- .rmpkg(package[keep])
##     }
##     pkgDesc &lt;- lapply(package, packageDescription, encoding = NA)
##     if (length(package) == 0) 
##         stop(&quot;no valid packages were specified&quot;)
##     basePkgs &lt;- sapply(pkgDesc, function(x) !is.null(x$Priority) &amp;&amp; 
##         x$Priority == &quot;base&quot;)
##     z$basePkgs &lt;- package[basePkgs]
##     if (any(!basePkgs)) {
##         z$otherPkgs &lt;- pkgDesc[!basePkgs]
##         names(z$otherPkgs) &lt;- package[!basePkgs]
##     }
##     loadedOnly &lt;- loadedNamespaces()
##     loadedOnly &lt;- loadedOnly[!(loadedOnly %in% package)]
##     if (length(loadedOnly)) {
##         names(loadedOnly) &lt;- loadedOnly
##         pkgDesc &lt;- c(pkgDesc, lapply(loadedOnly, packageDescription))
##         z$loadedOnly &lt;- pkgDesc[loadedOnly]
##     }
##     z$matprod &lt;- as.character(options(&quot;matprod&quot;))
##     es &lt;- extSoftVersion()
##     z$BLAS &lt;- as.character(es[&quot;BLAS&quot;])
##     z$LAPACK &lt;- La_library()
##     class(z) &lt;- &quot;sessionInfo&quot;
##     z
## }
## &lt;bytecode: 0x7fb16ea6e9e8&gt;
## &lt;environment: namespace:utils&gt;</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
