<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Javier Herrero" />


<title>Twitter coverage of the Biology of Genomes 2019 conference</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lazappi's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Meetings
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="bog2019.html">CHSL Biology of Genomes - May 2019</a>
    </li>
    <li>
      <a href="CancerDomainMay19.html">Rethinking Cancer - UCL Cancer Domain - May 2019</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/jherrero/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Twitter coverage of the Biology of Genomes 2019 conference</h1>
<h4 class="author"><em>Javier Herrero</em></h4>
<h4 class="date"><em>2019-05-13 11:55:23</em></h4>

</div>


<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>An analysis of tweets from the Biology of Genomes 2019 conference. A total of 2398 tweets from 677 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1">
<h1><span class="header-section-number">2</span> Timeline</h1>
<div id="tweets-by-day" class="section level2">
<h2><span class="header-section-number">2.1</span> Tweets by day</h2>
<p><img src="figures/bog2019/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2">
<h2><span class="header-section-number">2.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2019-05-07 - 2019-05-11 in the America/New_York timezone.</p>
<p><img src="figures/bog2019/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1">
<h1><span class="header-section-number">3</span> Users</h1>
<div id="top-tweeters" class="section level2">
<h2><span class="header-section-number">3.1</span> Top tweeters</h2>
<p><img src="figures/bog2019/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-original-tweeters" class="section level2">
<h2><span class="header-section-number">3.2</span> Top original tweeters</h2>
<p><img src="figures/bog2019/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-proportion" class="section level2">
<h2><span class="header-section-number">3.3</span> Retweet proportion</h2>
<p><img src="figures/bog2019/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-time" class="section level2">
<h2><span class="header-section-number">3.4</span> Top tweeters by time</h2>
<p><img src="figures/bog2019/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2">
<h2><span class="header-section-number">3.5</span> Top tweeters by day</h2>
<div id="all-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.1</span> All tweets</h3>
<div id="day-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/bog2019/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/bog2019/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/bog2019/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/bog2019/tweets-top-users-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5" class="section level4 unnumbered">
<h4>Day 5</h4>
<p><img src="figures/bog2019/tweets-top-users-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.2</span> Original tweets</h3>
<div id="day-1-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-1" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5-1" class="section level4 unnumbered">
<h4>Day 5</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="sources" class="section level2">
<h2><span class="header-section-number">3.6</span> Sources</h2>
<p><img src="figures/bog2019/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="networks" class="section level1">
<h1><span class="header-section-number">4</span> Networks</h1>
<div id="replies" class="section level2">
<h2><span class="header-section-number">4.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by page rank.</p>
<p><img src="figures/bog2019/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2">
<h2><span class="header-section-number">4.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Node colour, label size and alpha adjusted according to PageRank score.</p>
<p><img src="figures/bog2019/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets" class="section level1">
<h1><span class="header-section-number">5</span> Retweets</h1>
<div id="retweet-proportion-1" class="section level2">
<h2><span class="header-section-number">5.1</span> Retweet proportion</h2>
<p><img src="figures/bog2019/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-count" class="section level2">
<h2><span class="header-section-number">5.2</span> Retweet count</h2>
<p><img src="figures/bog2019/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-retweets" class="section level2">
<h2><span class="header-section-number">5.3</span> Top retweets</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Peak east coast techno-hipster cryptocurrency Americana cannabis genome #bog19 <a href="https://t.co/x4hOYHdeGe" class="uri">https://t.co/x4hOYHdeGe</a>
</td>
<td style="text-align:right;">
61
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Just in time for #bog19, <span class="citation">@genomeresearch</span> posts early access version (today) of our paper where we found 2250 bacterial genomes that had human contamination, plus 3,437 “proteins” that are erroneous translations of these human contaminants <a href="https://t.co/koQRkmLRPw" class="uri">https://t.co/koQRkmLRPw</a>
</td>
<td style="text-align:right;">
42
</td>
</tr>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
athmapai
</td>
<td style="text-align:left;">
Recent or soon to be grads: looking for a postdoc in gene regulation, RNA, and genomics? I’d love to chat! Come find me at #BoG19! I’ll be presenting poster #185 on Thursday <a href="https://t.co/FBsPTVH0Tn" class="uri">https://t.co/FBsPTVH0Tn</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
Do you know how many questions have been asked since the last one asked by a woman? I do. It’s 32. #BoG19
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
JedMSP
</td>
<td style="text-align:left;">
Oxford Nanopour #bog19 <a href="https://t.co/QeSusqSoaz" class="uri">https://t.co/QeSusqSoaz</a>
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
hagentilgner
</td>
<td style="text-align:left;">
Wow: ScISOr-Seq type work is catching on (<a href="https://t.co/JVU0YLcLLQ" class="uri">https://t.co/JVU0YLcLLQ</a>). Happy to see this and absolutely have to read in detail. BTW, if you like single-cell isoform work (see <a href="https://t.co/OFPcZDaxMt" class="uri">https://t.co/OFPcZDaxMt</a>), come and see me at poster 271. We are hiring! #BoG19
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Anonymous questions from the audience (moderated) are a terrific way to include more audience members. Otherwise the questions tend to be from a small cadre of frequent-askers. Plus it allows people to phrase their questions more carefully #BoG19 <a href="https://t.co/TOFIHMVlog" class="uri">https://t.co/TOFIHMVlog</a>
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
For those of you interested in the experiment we’re doing at #BoG19, here’s some preliminary results (right before the last talk): Participation, <em>in person</em>, has changed drastically. For the first time ever, we’ve accidentally created overrepresentation. <a href="https://t.co/3Zn0JSDFFS" class="uri">https://t.co/3Zn0JSDFFS</a>
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
aaronquinlan
</td>
<td style="text-align:left;">
Really fun talk from <span class="citation">@BeEngelhardt</span> on a Bayesian approach for optimizing experimental design to maximize cell type discovery from single-cell sequencing experiments. Calling it GT-TS (Good-Toulmin like estimator via Thompson sampling). Paper here: <a href="https://t.co/uWm7ySP4lG" class="uri">https://t.co/uWm7ySP4lG</a> #bog19
</td>
<td style="text-align:right;">
14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="favourites" class="section level1">
<h1><span class="header-section-number">6</span> Favourites</h1>
<div id="favourite-proportion" class="section level2">
<h2><span class="header-section-number">6.1</span> Favourite proportion</h2>
<p><img src="figures/bog2019/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="favourite-count" class="section level2">
<h2><span class="header-section-number">6.2</span> Favourite count</h2>
<p><img src="figures/bog2019/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-favourites" class="section level2">
<h2><span class="header-section-number">6.3</span> Top favourites</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Peak east coast techno-hipster cryptocurrency Americana cannabis genome #bog19 <a href="https://t.co/x4hOYHdeGe" class="uri">https://t.co/x4hOYHdeGe</a>
</td>
<td style="text-align:right;">
231
</td>
</tr>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
203
</td>
</tr>
<tr>
<td style="text-align:left;">
JedMSP
</td>
<td style="text-align:left;">
Oxford Nanopour #bog19 <a href="https://t.co/QeSusqSoaz" class="uri">https://t.co/QeSusqSoaz</a>
</td>
<td style="text-align:right;">
152
</td>
</tr>
<tr>
<td style="text-align:left;">
jtung5
</td>
<td style="text-align:left;">
At the #BoG19 closing banquet and the big news is that <span class="citation">@blekhman</span> just received tenure!! Big congrats Ran, so well deserved!! <a href="https://t.co/9s8c9bMgdX" class="uri">https://t.co/9s8c9bMgdX</a>
</td>
<td style="text-align:right;">
117
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Just in time for #bog19, <span class="citation">@genomeresearch</span> posts early access version (today) of our paper where we found 2250 bacterial genomes that had human contamination, plus 3,437 “proteins” that are erroneous translations of these human contaminants <a href="https://t.co/koQRkmLRPw" class="uri">https://t.co/koQRkmLRPw</a>
</td>
<td style="text-align:right;">
82
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Anonymous questions from the audience (moderated) are a terrific way to include more audience members. Otherwise the questions tend to be from a small cadre of frequent-askers. Plus it allows people to phrase their questions more carefully #BoG19 <a href="https://t.co/TOFIHMVlog" class="uri">https://t.co/TOFIHMVlog</a>
</td>
<td style="text-align:right;">
79
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
Do you know how many questions have been asked since the last one asked by a woman? I do. It’s 32. #BoG19
</td>
<td style="text-align:right;">
76
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
#bog19 the first BOG without the portrait of Watson. Finally. <a href="https://t.co/i42jfyMKdt" class="uri">https://t.co/i42jfyMKdt</a>
</td>
<td style="text-align:right;">
63
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
For those of you interested in the experiment we’re doing at #BoG19, here’s some preliminary results (right before the last talk): Participation, <em>in person</em>, has changed drastically. For the first time ever, we’ve accidentally created overrepresentation. <a href="https://t.co/3Zn0JSDFFS" class="uri">https://t.co/3Zn0JSDFFS</a>
</td>
<td style="text-align:right;">
61
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Love it that the genome scaffolders used by <span class="citation">@mike_schatz</span> for the tomato genome are called SALSA2 and RaGOO. Coincidence? #BoG19
</td>
<td style="text-align:right;">
60
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="quotes" class="section level1">
<h1><span class="header-section-number">7</span> Quotes</h1>
<div id="quote-proportion" class="section level2">
<h2><span class="header-section-number">7.1</span> Quote proportion</h2>
<p><img src="figures/bog2019/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="quote-count" class="section level2">
<h2><span class="header-section-number">7.2</span> Quote count</h2>
<p><img src="figures/bog2019/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-quotes" class="section level2">
<h2><span class="header-section-number">7.3</span> Top quotes</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
AnnaCuomo92
</td>
<td style="text-align:left;">
Great Illustration and great meeting so far! #BoG19 <a href="https://t.co/MbRJgVUS5E" class="uri">https://t.co/MbRJgVUS5E</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
Awesomics
</td>
<td style="text-align:left;">
The Biology of Genomes (#BoG19) now underway <a href="https://t.co/uJpygePI9e" class="uri">https://t.co/uJpygePI9e</a>. Access live streamed talks, abstract PDF, and video archive (for 6 weeks) at <a href="https://t.co/L8JiHY0BFH" class="uri">https://t.co/L8JiHY0BFH</a> and enter “99” in the ‘Payment Authorized’ field. Cost=$99. Thanks <span class="citation">@cshlmeetings</span>! <a href="https://t.co/a4zUxtCduV" class="uri">https://t.co/a4zUxtCduV</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
deboramarks
</td>
<td style="text-align:left;">
Club #BoG19 ever wants me as a ‘member’ I’d pretend I didn’t want to join but then I’d say yes within 30 seconds-sounds like a great conf <a href="https://t.co/8hW8w9NCcf" class="uri">https://t.co/8hW8w9NCcf</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
mammothbiosci
</td>
<td style="text-align:left;">
Exciting to hear about everything going on at #BoG19 - work in #genome #biology helps drive the creation of DNA-based diagnostic tools. <a href="https://t.co/9kf9P9T051" class="uri">https://t.co/9kf9P9T051</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
DevilsAdvoMax
</td>
<td style="text-align:left;">
Big, big thanks <span class="citation">@NatalieTelis</span> and Friday chairs for pushing towards participatory diversity in the QAs. Clear improvements over the course of #BoG19 sessions - a more inviting spirit for young trainees, first-time attendees or simply more hesitant folks. Exemplary activism! 💕🙂 <a href="https://t.co/G1rrlNS8bB" class="uri">https://t.co/G1rrlNS8bB</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
girlscientist
</td>
<td style="text-align:left;">
Interesting – here’s what offering the chance to ask questions online during a talk did to in-person questions at #BoG19 (thread)…. <a href="https://t.co/zzr6KPoOer" class="uri">https://t.co/zzr6KPoOer</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="media" class="section level1">
<h1><span class="header-section-number">8</span> Media</h1>
<div id="media-count" class="section level2">
<h2><span class="header-section-number">8.1</span> Media count</h2>
<p><img src="figures/bog2019/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-media" class="section level2">
<h2><span class="header-section-number">8.2</span> Top media</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Peak east coast techno-hipster cryptocurrency Americana cannabis genome #bog19 <a href="https://t.co/x4hOYHdeGe" class="uri">https://t.co/x4hOYHdeGe</a>
</td>
<td style="text-align:right;">
231
</td>
</tr>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
203
</td>
</tr>
<tr>
<td style="text-align:left;">
JedMSP
</td>
<td style="text-align:left;">
Oxford Nanopour #bog19 <a href="https://t.co/QeSusqSoaz" class="uri">https://t.co/QeSusqSoaz</a>
</td>
<td style="text-align:right;">
152
</td>
</tr>
<tr>
<td style="text-align:left;">
jtung5
</td>
<td style="text-align:left;">
At the #BoG19 closing banquet and the big news is that <span class="citation">@blekhman</span> just received tenure!! Big congrats Ran, so well deserved!! <a href="https://t.co/9s8c9bMgdX" class="uri">https://t.co/9s8c9bMgdX</a>
</td>
<td style="text-align:right;">
117
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
#bog19 the first BOG without the portrait of Watson. Finally. <a href="https://t.co/i42jfyMKdt" class="uri">https://t.co/i42jfyMKdt</a>
</td>
<td style="text-align:right;">
63
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
For those of you interested in the experiment we’re doing at #BoG19, here’s some preliminary results (right before the last talk): Participation, <em>in person</em>, has changed drastically. For the first time ever, we’ve accidentally created overrepresentation. <a href="https://t.co/3Zn0JSDFFS" class="uri">https://t.co/3Zn0JSDFFS</a>
</td>
<td style="text-align:right;">
61
</td>
</tr>
<tr>
<td style="text-align:left;">
aaronquinlan
</td>
<td style="text-align:left;">
Are you interested in what can be learned about germline mutation dynamics from WGS of large, 3-generation pedigrees? Check out <span class="citation">@tomsasani</span>’s poster (240) today. Tom is exceptional and is looking for postdoc opps in the coming year. Preprint: <a href="https://t.co/CwpBnGP0f1" class="uri">https://t.co/CwpBnGP0f1</a> #bog19 <a href="https://t.co/DkbX1KqHPh" class="uri">https://t.co/DkbX1KqHPh</a>
</td>
<td style="text-align:right;">
58
</td>
</tr>
<tr>
<td style="text-align:left;">
kajawasik
</td>
<td style="text-align:left;">
Whenever I’m back at <span class="citation">@CSHL</span> I realize how much I miss this place. It really made me a scientist and taught me to enjoy the small things in life like wine and cheese on President’s lawn.. #bog19 <a href="https://t.co/efXS121Mh9" class="uri">https://t.co/efXS121Mh9</a>
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
NeBanovich
</td>
<td style="text-align:left;">
Fish mafia #bog19 #GenomesSparkJoy <a href="https://t.co/yEgLgcQlAx" class="uri">https://t.co/yEgLgcQlAx</a>
</td>
<td style="text-align:right;">
40
</td>
</tr>
<tr>
<td style="text-align:left;">
notSoJunkDNA
</td>
<td style="text-align:left;">
#bog19 “Germline and somatic variant calling with NovaSeq™ 6000 2x250bp reads” <span class="citation">@nygenome</span> poster with <span class="citation">@mjs2225</span> <span class="citation">@gnarzisi</span> et al. To be presented on Friday but available at <a href="https://t.co/tETp7Brcmz" class="uri">https://t.co/tETp7Brcmz</a> <a href="https://t.co/Xky5B00oXy" class="uri">https://t.co/Xky5B00oXy</a>
</td>
<td style="text-align:right;">
34
</td>
</tr>
</tbody>
</table>
<div id="most-liked-media-image" class="section level3">
<h3><span class="header-section-number">8.2.1</span> Most liked media image</h3>
<div class="figure">
<img src="http://pbs.twimg.com/media/D6Je2pUXoAUjpUq.jpg" />

</div>
</div>
</div>
</div>
<div id="tweet-text" class="section level1">
<h1><span class="header-section-number">9</span> Tweet text</h1>
<div id="word-cloud" class="section level2">
<h2><span class="header-section-number">9.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/bog2019/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2">
<h2><span class="header-section-number">9.2</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 5 times.</p>
<p><img src="figures/bog2019/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2">
<h2><span class="header-section-number">9.3</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/bog2019/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset">
<h3><span class="header-section-number">9.3.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered">
<h4>Topic 1</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
.<span class="citation">@nonfiniteloop</span> — Can we detect post zygotic variations. Re-purpose GTEx data. Develop method to detect mutations from bulk RNA-seq and validate in Sun exposed/unexposed skin. More mutations in UV exposed skin, increase in UV-induced mut. signature #BoG19
</td>
<td style="text-align:right;">
0.9936772
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Elise <span class="citation">@EliseScience</span> thinks that the + and - correlated eQTLs with aFC could be due to transcription factor binding levels</p>
<p>Correlated aFC with TF expression in each tissue (used ENCODE data)</p>
<p>5.6% of tested TF-eQTLs pairs are corr with 1 TF 41% of are corr with &amp;gt; 1TF</p>
#BoG19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Julie Jerber, mapping neuronal development eQTLs. eQTL limitations: tissue, time points, stimulations not feasible (in vivo). Instead use hiPS cells. Go beyond primary blood cells! using iPS can stimulate and then analyze at single cell level (capture different cells). #BoG19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Calvert/2: single cell analysis enables detection of infiltrating cells in various pathological conditions of human brains. 27,696 patients genes tested across 96 patients, analyzed with Rasqual, -&amp;gt; 4382 eQTL genes (Including known disease candidate genes). #BoG19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
SM: acquired genomic mosaicism most apparent in blood-derived DNA (small number of SNPs that have unusual small allelic fraction in exome sequencing). Clonal hematopoiesis rarely detected in young individual, process takes decades to make up sig fraction of cell pop. #BoG19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Wu/2 capitalize on exome sequencing, predict candidate new antigens. POC: Melanoma, predicting antigens, manufacturing antigens, given to patients with adjuvant to stimulate immune memory. Other study-glioblastoma. Combination with anti- PD-1. Successful promising so far. #BoG19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Rockweiler/2: measuring skin exposed/not to UVs showing more skin mutation in caucasians if compared to African American donors. identify 79 mutations/donor, 4/tissues, 7% overlap, &amp;gt;82-88% validation rate. Mutation rate very variable across tissues and samples. #bog19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Jerber/2: inducing dopaminergic neurons (3 time points, 11-30-52 days) and perturbations (toxicity drugs: rotenone); from 129 donors; Capturing nicely the cell types. Rotenone treatment shows clear gene expression differences -&amp;gt; discovers eQTLs for all cell types. #BoG19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Rockweiler/3. Mutations are tissue specific (example: C-&amp;gt;T in skin). Brain less mutations. Age/timing? Shared mutation is more ancient -&amp;gt; Surprise: 80% of mutations happen between zygote and gastrula! (surprise: in so few cell cycles!) Do not miss cell specific ones? #bog19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Fiona Calvert; regulatory map of human primary microglia (~resident macrophages of CNS). Why important? -&amp;gt; genetics data points at association with Alzheimers. Single cell sequencing (15 patients) -&amp;gt; association with neurological disorders. Here 141 neurosurgery patients. #BoG19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered">
<h4>Topic 2</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Farren Isaacs; Multiple automated genome engineering (MAGE) works without double strand brake (big advantage). Eukaryotic MAGE. Using a ssODN annealing DNA replication fork enhances gene editing 100x. Multiplexing: msh2- allow multiple editing events. Cycling adds variants #bog19
</td>
<td style="text-align:right;">
0.9938864
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>EG is using 12 pcw (post conception weeks) human neural stem cells with a CRISPR/Cas9 deletion screen to learn more about enhancer function (2,233 of them) across 26,260 non-coding regions and 10,656 coding regions expressed in this cell line, + 2,608 bkg regs via ROADMAP</p>
#BoG19
</td>
<td style="text-align:right;">
0.9936772
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Sanders: Structural variations in single cells. Developed a method for single cell genome sequencing which maintains strand information, taking advantage of BdrU incorporation -&amp;gt; digestion of one strand -&amp;gt; sequencing the remaining strand. Measures depth, strand, phase. #BoG19
</td>
<td style="text-align:right;">
0.9936772
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Ashley Sanders developed Strand-seq which takes advantage of the paternal homologues to track cells across divisions</p>
<p><a href="https://t.co/K3xs3OcG5I" class="uri">https://t.co/K3xs3OcG5I</a> abstract: “resolves the individual homologs within a cell”</p>
<p>scTRIP: single cell Tri-channel processing is the follow up method</p>
#BoG19 <a href="https://t.co/jKG5JgFcfW" class="uri">https://t.co/jKG5JgFcfW</a>
</td>
<td style="text-align:right;">
0.9932127
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>YQ from <span class="citation">@MarthGabor</span> wants to study subclone-specific mutations, #scRNAseq fits the task very well</p>
<p>Challenge: cell coverage is low, roughly 3 MB of “real-estate” for determining the presence of somatic variants, so only ~20 somatic variants with coverage</p>
<ul>
<li>allele drop-out #BoG19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Luca Pinello <span class="citation">@lucapinello</span> CRISPResso (cool name!): analysis of CRISPR-Cas genome editing outcomes from deep sequencing! Now CRISPResso2! Which will take into account base modification, batch mode, allele specific quantification, biology informed alignment, fast processing! #bog19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Ines Cebola, gene regulation in pancreas islets releated to type2 diabetes (T2D). GWAS previously identified; genomics (many experiments) identifies “regulome”, mapping enhancers etc. Next: how to assign enhancers to true targets; Can we understand molecular mechanisms. #bog19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Hannah Currant, on UK Biobank, 500K individuals, 96M variants: focus on retinal phenotype. Retina inspected for physical differences (example: thickness, as “endophenotype”), which is connected with diseases/biology. ~31K GWAS- identify loci associated with retina problems #bog19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
<tr>
<td style="text-align:left;">
cshlmeetings
</td>
<td style="text-align:left;">
Unable to make it to next week’s #BoG19? Live stream it! Each $99 subscription includes live stream of talks, PDF of the abstract book, and six-week access to #BoG19’s video archive. Register online: <a href="https://t.co/fMcgEzbvys" class="uri">https://t.co/fMcgEzbvys</a> and enter “99” in the ‘Payment Authorized’ field. <a href="https://t.co/AHcQxt1Sg6" class="uri">https://t.co/AHcQxt1Sg6</a>
</td>
<td style="text-align:right;">
0.9923720
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Evan Geller with the fascinating question of what makes human brains unique. Mutations of regulatory elements (enhancers) a better candidate than mutations in coding regions. Then experiments to mutagenize regulatory elements in human neural stem cells. #BoG19
</td>
<td style="text-align:right;">
0.9923720
</td>
</tr>
</tbody>
</table></li>
</ul>
</div>
<div id="topic-3" class="section level4 unnumbered">
<h4>Topic 3</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
JMR: Settlement of Near / Remote Oceania - modern humans arrived in Near Oceania ~40-65kya, but did not extend beyond Soloman island. Austronesian expansion from Taiwan ~ 5.5 kya; Lapita people arrived in Vanuatu ~3kya; mirgration from Bismarck Archipelago ~2.5kya. #BoG19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
JX: patients hospitalized for bone marrow transplantation provide longitudinal data for host-microbiome studies. For ~2 weeks patients are immunocompromised, hence carefully monitored. Has a lot of clinical variable/meta-data, and can collect stool sample for microbiome. #BoG19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Peter <span class="citation">@genemodeller</span> on types of joint analysis:</p>
<ul>
<li>meta-analysis</li>
<li>fine-mapping</li>
<li>PheWAS-analysis</li>
</ul>
<p>You could also cluster GWAS results. Did so using a bayesian approach (would make <span class="citation">@BeEngelhardt</span> proud ^^) based on a Dirichlet prior</p>
<p>Can analyze simultaneously &amp;gt;1k GWAS</p>
#BoG19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
<tr>
<td style="text-align:left;">
obahcall
</td>
<td style="text-align:left;">
Appreciation for our wonderful #BoG19 organizers, who are also illustrious scientists celebrating recent well deserved recognitions!<br />
Matt Hurles <span class="citation">@mehurles</span> elected FRS <span class="citation">@royalsociety</span> Elaine Ostrander elected <span class="citation">@theNASciences</span> Jay Shendure <span class="citation">@JShendure</span> Lounsbery award <span class="citation">@theNASciences</span>
</td>
<td style="text-align:right;">
0.9923720
</td>
</tr>
<tr>
<td style="text-align:left;">
saketkc
</td>
<td style="text-align:left;">
My poster with <span class="citation">@WenzhengLi</span> and Andrew Smith for #bog19! We developed a simple method for finding actively-translating ORFs from Ribo-seq data. Performs well at both short and long ORFs. Code: <a href="https://t.co/s9Z77yGD2T" class="uri">https://t.co/s9Z77yGD2T</a> (also on pip and bioconda) Poster: <a href="https://t.co/rKwfLQQVbI" class="uri">https://t.co/rKwfLQQVbI</a>. <a href="https://t.co/gXdXAPqpEC" class="uri">https://t.co/gXdXAPqpEC</a>
</td>
<td style="text-align:right;">
0.9923720
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
MS: Good final chromosome level leveraging reference and HiC for validation, polishing with short reads using FreeBayes as the polisher. Shows an example of the “easy to break free” tomato (jointless) and “long stem” tomato which has a negative epistatic interaction #BoG19
</td>
<td style="text-align:right;">
0.9920435
</td>
</tr>
<tr>
<td style="text-align:left;">
sorrywm
</td>
<td style="text-align:left;">
JXavier finds that gut microbial diversity is predictive of recovery from bone marrow transplants, uses modeling in model systems to identify inter-microbial interactions, and tries out recovering a patient’s microbiome using their own pre-operation fecal sample. #BoG19
</td>
<td style="text-align:right;">
0.9920435
</td>
</tr>
<tr>
<td style="text-align:left;">
NatureEcoEvo
</td>
<td style="text-align:left;">
<p>A taste of recent pop gen papers published in NEE:</p>
<p>Admixture with a ghost lineage in bonobos <a href="https://t.co/YGP2Ci0P9I" class="uri">https://t.co/YGP2Ci0P9I</a></p>
<p>Using selection at linked sites to estimate adaptation <a href="https://t.co/RRn3kmVmju" class="uri">https://t.co/RRn3kmVmju</a></p>
Ancient genomes of Neolithic Britain <a href="https://t.co/OXJwkj7hbn" class="uri">https://t.co/OXJwkj7hbn</a> #BoG19
</td>
<td style="text-align:right;">
0.9920435
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Next up Anat Kreimer <span class="citation">@anatkreimer</span> from Nir Yosef’s lab on using massively parallel reporter assays (MPRAs) for high-throughput enhancer validation (uses a lentivirus that is transducted, then followed by RNA and DNA seq)</p>
<p>Related pre-print: <a href="https://t.co/UeaaDYWm6U" class="uri">https://t.co/UeaaDYWm6U</a></p>
#bog19
</td>
<td style="text-align:right;">
0.9920435
</td>
</tr>
<tr>
<td style="text-align:left;">
NeBanovich
</td>
<td style="text-align:left;">
RB: Describing why it is so difficult to quantify the heritability of the gut microbiome in humans - very plastic trait that is strongly impacted by dietary choices - decided to use the Amboseli Baboons - near-daily observation and fecal samples collected since 2000. #bog19
</td>
<td style="text-align:right;">
0.9916855
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered">
<h4>Topic 4</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>SMAC-seq by Georgi Marinov and team re-captures the peaks by DNA-seq peak but also overlaps H4S37C chemical nucleosome mapping peaks</p>
<p>Their profiles are nicely phased with Cse4 ChIP-seq as well</p>
<p>(I don’t know what Cse4 nor H4S37C do! Tweet and later revisit? :P)</p>
#bog19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
nanopore
</td>
<td style="text-align:left;">
Will you be at Biology of Genomes next week? Register for our free lunch &amp;amp; seminar on Thursday 9th May. Steven Salzberg is giving a talk on ‘The Redwood Genome Project: sequencing and assembling the largest trees in the world’ join us here: <a href="https://t.co/gq8U4SXRh8" class="uri">https://t.co/gq8U4SXRh8</a> #bog19 <a href="https://t.co/GTbHxxK06n" class="uri">https://t.co/GTbHxxK06n</a>
</td>
<td style="text-align:right;">
0.9923720
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
MS: greedily identify the most diverse species from short-read data to identify candidates for long-read sequencing. PromethION improved sequencing throughput dramatically. Read length ~10-100kb length, rarily &amp;gt; 1 Mb. #BoG19
</td>
<td style="text-align:right;">
0.9923720
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
MS: Need to bring some new technology to capture SVs. there are 50 varieties with short reads from 10 main clades. Long Reads from <span class="citation">@nanopore</span> - start at MinION+GridION - 5/10 flowcells per genome - but then PromethION really changed the game. Best run 140GB flow cell #BoG19
</td>
<td style="text-align:right;">
0.9920435
</td>
</tr>
<tr>
<td style="text-align:left;">
PLOSBiology
</td>
<td style="text-align:left;">
John Wolters <span class="citation">@HittingerLab</span>: Engineering yeast mtDNA: deplete mtDNA in recipient, put manipulated mtDNA back with gene gun, prime translation, go! Use small, less AT-rich Naumovozyme castellii backbone with S. cerevisiae genes patched in. Ongoing… #BoG19
</td>
<td style="text-align:right;">
0.9920435
</td>
</tr>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
Ready for #BoG19. Poster tube to present the work of Cecilia (bright BSc student) and Garima (bright postdoc) on using vg for somatic variant detection. All started while chatting to <span class="citation">@erikgarrison</span> last year at #BoG18. At the airport, free coffee refills. Will need them! 😎
</td>
<td style="text-align:right;">
0.9916855
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
KL: Otters diverged from terrestrial mustelids ~14 Mya, Sea otter diverged from giant otter ~10 Mya. Giant otter genome sequenced at Broad already, here sequence the Sea otter genome. #BoG19
</td>
<td style="text-align:right;">
0.9912937
</td>
</tr>
<tr>
<td style="text-align:left;">
obahcall
</td>
<td style="text-align:left;">
So glad to see #BoG19 continues to lead in the genome way. (Nearly) all talks have been tweetable. And most speakers encourage: showing twitter handle, relevant preprints, software &amp;amp; QR code. Discussion in room &amp;amp; online goes much deeper from this- esp when we can read preprint.
</td>
<td style="text-align:right;">
0.9912937
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
RF: Talks us about spermatid biology which form syncetium between gametes (!) - freely sharing the mRNA between spermatids but some genes can be selfish - this is the explanation for the Mouse T complex genetics - lead to extreme transmission distortion. #bog19
</td>
<td style="text-align:right;">
0.9912937
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Cristoph Bock on CISRP-single cell sequencing. High throughput functional biology! To establish causality in epigenetics- without using ONE GENE - ONE POSTDOC, which is very risky on postdoc career, unless you’re conservative!! -&amp;gt; arguments for the need high throughput! #bog19
</td>
<td style="text-align:right;">
0.9912937
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-5" class="section level4 unnumbered">
<h4>Topic 5</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Joseph Thornton used an ordinal logistic regression model to study epistasis effects and main effects</p>
<p>A joint model includes 2 extra sum of coefficients (sum of i to j)</p>
<p>Misclassification rate is &amp;lt; 0.6% overall</p>
<p>Presented a heatmap of main vs epistasis significant terms</p>
#BoG19
</td>
<td style="text-align:right;">
0.9932127
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
AH: Performed matched unrelated-GWAS vs sib-GWAS to look at this “three headed dragon”. Can plot sib-GWAS vs unrelated. Thankfully physical traits line up pretty well, but social traits (educational attainment, smoking, age at first sex) show <em>strong</em> shifts #bog19
</td>
<td style="text-align:right;">
0.9932127
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
DV: PRS by either GW SNPs, conditionally independent SNPs, fine-mapped SNPs, etc. Effect size from UKB, validated in INTERVAL study (N ~ 43K). Measure of prediction accuracy is Pearson’s R - actually observed improvement with decreasing number of variants from 5M to 300) #BoG19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
RJ: Causal effect size distribution parameterized by effect size (beta) and number of causal SNPs (k) (Zhang et al. NG 2018). Want to know how is heritability divided - centered on 1 SNP or distributed among multiple SNPs, has implication to effect distr at regional level. #BoG19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
HM: male-bias not explained by de novo variants found on X chr. Haldane proposed a theory of de novo vs. inherited pathogenic variants in X-linked recessive genes, based on mutation-sel balanec. Expect 18%, but observed 32% #BoG19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
DV: BCX consortium (N &amp;gt; 560K EUropean ancestry meta-analysis) found &amp;gt; 10,000 variants associated with at least 1 of 29 blood count phenotypes, 510 rare (&amp;lt; 1% MAF). Statistical fine-mapping assign 10% of the loci to a single variant. #BoG19
</td>
<td style="text-align:right;">
0.9929538
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
BP: observation “SNP with lower levels of LD have significantly larger per-SNP heritability” - MAF/LD-dependent architecture biases SNP heritability estimation. Today will discuss assumptions of estimation, and how to drop them to estimate SNP h2. #BoG19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
AH: under null, prediction accuracy should be equal between GWAS and sib-GWAS. If there are indirect effects, GWAS-based PS should outperform sib-GWAS-based. Find for most traits standard GWAS outperforms (matched sample size between GWASes) #BoG19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
AH: what to investigate if other factors beyond ancestry impacts PS. Used the “White British” of UKB. Performed GWAS in 120K men + 120K women, and then tested prediction accuracy in 20K men and 20K women, separately. Prediction accuracy differed by sex. #BoG19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Now Hanna Ollila <span class="citation">@hannamollila</span> from <span class="citation">@jkpritch</span>’s lab (his lab meetings must feel like today’s morning session hehe) on USF1 as a pleiotropic meta regular <em>for sleep and circadian rhythms</em></p>
<p>Story started back in her PhD in Helsinki</p>
<p><a href="https://t.co/cVT9ZwKbbc" class="uri">https://t.co/cVT9ZwKbbc</a></p>
#BoG19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered">
<h4>Topic 6</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
majbvc
</td>
<td style="text-align:left;">
El <span class="citation">@centrodemocratico</span> buscará un candidato distinto a Ángela Garzón para enfrentar las elecciones en Bogotá. Será…??? O será otra salida en falso…?? #losleo <span class="citation">@AlvaroUribeVel</span> <span class="citation">@diegomolanovega</span> <span class="citation">@SamuelHoyosM</span> #FelizJueves #BoG19 #filbo2019 #video <span class="citation">@ChurchillBogota</span> <span class="citation">@MiguelUribeT</span> <a href="https://t.co/GUJ8jaznOy" class="uri">https://t.co/GUJ8jaznOy</a>
</td>
<td style="text-align:right;">
0.9936772
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
JH: chromatin-modifying genes (CMBs) - diverse set of genes that bind chromatin and regulate cellular processes (remodelling or modification). In DDD project (13.6k patient with undiagnosed DD) de novo variants explain ~50% of risk, enriched in CMG set. #BoG19
</td>
<td style="text-align:right;">
0.9936772
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Marina Ruiz-Romero (cc <span class="citation">@GuigoLab</span>) has ~50 million Illumina TruSeq 75 bp reads from 9 fly samples across 3 developmental stages</p>
<p>She found that tissue specificity increases along differentiation from larva 3 to white pupa to fly (?)</p>
<p>matches modEncode data based on PC1/2</p>
#BoG19
</td>
<td style="text-align:right;">
0.9932127
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Coronary artery disease is the leading global cause of mortality</p>
<p>Current risk assessment of heart attack reviewed by Khera et al <a href="https://t.co/7BJCbpN2xv" class="uri">https://t.co/7BJCbpN2xv</a> based on non-🧬metrics</p>
but <span class="citation">@skathire</span> has now shown that polygenic risk score is the first risk factor for this disease #BoG19 <a href="https://t.co/UuHPODF7cq" class="uri">https://t.co/UuHPODF7cq</a>
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
JX: can we explain sudden shifts in microbial composition? In mice, single dose of clindamycin lead to reduction of diversity, highly variable over time, but eventually by day 21 or so stabilize to a lower diversity composition. Replicated in multiple animals #BoG19
</td>
<td style="text-align:right;">
0.9926744
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Mikkel <span class="citation">@mikkelschierup</span> is talking about Skov et al, bioRxiv, 2018 <a href="https://t.co/modrYeXWWz" class="uri">https://t.co/modrYeXWWz</a></p>
<p>They can use this to derive mean archaic segmenting lengths (SGDP) &amp;amp; the total amount of archaic sequence</p>
<p>Then they explored how the segments are shared by West Eurasia &amp;amp; East Asia</p>
#BoG19 <a href="https://t.co/XSh7Ah1kmI" class="uri">https://t.co/XSh7Ah1kmI</a>
</td>
<td style="text-align:right;">
0.9923720
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
JMR: exploratory PCA using archaic samples show Denisovan cline for Oceanian population. Neandertal anc ~uniformly distributed (~2-3%), but Denisovan anc more variable (0-3%), and scale well with Papuan ancestry (except some outlier population like Agta HG in Phillipines. #BoG19
</td>
<td style="text-align:right;">
0.9923720
</td>
</tr>
<tr>
<td style="text-align:left;">
GenomeNathan
</td>
<td style="text-align:left;">
<p><span class="citation">@adamauton</span> <span class="citation">@jkpritch</span> <span class="citation">@molly_przew</span> Don’t worry, you didn’t miss anything tonight – save two keynotes that anchored fresh insights on basic biological processes (mutation, development) in deep histories of the subfields in question.</p>
Basically, giants now…lucidly extending the footsteps of giants past. #bog19
</td>
<td style="text-align:right;">
0.9923720
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Peter <span class="citation">@genemodeller</span> wants to build polygenic risk scores for many traits using their fine mapping approach</p>
<p>He looked at the top 3, lowest 3 and middle 20% risk scores with the weights applied to <span class="citation">@uk_biobank</span> data. The results can inform policies for when to do screenings</p>
#BoG19
</td>
<td style="text-align:right;">
0.9923720
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
JX: Typical profile - diverse microbiome initially, then antibiotic dosed -&amp;gt; reduction of diversity and expansion of E. Coli -&amp;gt; body temperature increase -&amp;gt; switch of antibiotic -&amp;gt; solves problem, different microbe expanded. #BoG19
</td>
<td style="text-align:right;">
0.9923720
</td>
</tr>
</tbody>
</table>
<hr />
</div>
</div>
</div>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1>Session info</h1>
<pre><code>## R version 3.5.1 (2018-07-02)
## Platform: x86_64-apple-darwin15.6.0 (64-bit)
## Running under: macOS  10.14.4
## 
## Matrix products: default
## BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
## LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] en_GB.UTF-8/en_GB.UTF-8/en_GB.UTF-8/C/en_GB.UTF-8/en_GB.UTF-8
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] bindrcpp_0.2.2     here_0.1           knitr_1.20        
##  [4] viridis_0.5.1      viridisLite_0.3.0  ggraph_1.0.2      
##  [7] wordcloud_2.6      RColorBrewer_1.1-2 ggrepel_0.8.0     
## [10] ggplot2_3.0.0      igraph_1.2.2       topicmodels_0.2-8 
## [13] tidytext_0.2.0     forcats_0.3.0      lubridate_1.7.4   
## [16] tidyr_0.8.1        dplyr_0.7.6        rtweet_0.6.8      
## 
## loaded via a namespace (and not attached):
##  [1] Rcpp_0.12.19      lattice_0.20-35   utf8_1.1.4       
##  [4] assertthat_0.2.0  rprojroot_1.3-2   digest_0.6.18    
##  [7] ggforce_0.1.3     slam_0.1-45       R6_2.3.0         
## [10] plyr_1.8.4        backports_1.1.2   stats4_3.5.1     
## [13] evaluate_0.12     highr_0.7         httr_1.3.1       
## [16] pillar_1.3.0      rlang_0.2.2       lazyeval_0.2.1   
## [19] Matrix_1.2-14     rmarkdown_1.10    labeling_0.3     
## [22] stringr_1.3.1     munsell_0.5.0     broom_0.5.0      
## [25] compiler_3.5.1    janeaustenr_0.1.5 pkgconfig_2.0.2  
## [28] htmltools_0.3.6   tidyselect_0.2.5  tibble_1.4.2     
## [31] gridExtra_2.3     fansi_0.4.0       crayon_1.3.4     
## [34] withr_2.1.2       MASS_7.3-50       SnowballC_0.5.1  
## [37] grid_3.5.1        nlme_3.1-137      jsonlite_1.5     
## [40] gtable_0.2.0      magrittr_1.5      units_0.6-1      
## [43] scales_1.0.0      tokenizers_0.2.1  cli_1.0.1        
## [46] stringi_1.2.4     farver_1.0        reshape2_1.4.3   
## [49] NLP_0.2-0         xml2_1.2.0        tools_3.5.1      
## [52] glue_1.3.0        tweenr_1.0.0      purrr_0.2.5      
## [55] parallel_3.5.1    yaml_2.2.0        tm_0.7-6         
## [58] colorspace_1.3-2  bindr_0.1.1       modeltools_0.2-22</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
