<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Javier Herrero" />


<title>Twitter coverage of the Biology of Genomes 2019 conference</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.0.13/css/fa-svg-with-js.css" rel="stylesheet" />
<script src="site_libs/font-awesome-5.0.13/js/fontawesome-all.min.js"></script>
<script src="site_libs/font-awesome-5.0.13/js/fa-v4-shims.min.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->




<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}


.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
  padding-left: 25px;
  text-indent: 0;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>

<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">lazappi's Twitter stats</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Hashtags
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="bog2019.html">BoG19</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="https://github.com/jherrero/twitter-stats">
    <span class="fa fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Twitter coverage of the Biology of Genomes 2019 conference</h1>
<h4 class="author"><em>Javier Herrero</em></h4>
<h4 class="date"><em>2019-05-13 08:37:11</em></h4>

</div>


<div id="introduction" class="section level1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>An analysis of tweets from the Biology of Genomes 2019 conference. A total of 2394 tweets from 675 users were collected using the <code>rtweet</code> R package.</p>
</div>
<div id="timeline" class="section level1">
<h1><span class="header-section-number">2</span> Timeline</h1>
<div id="tweets-by-day" class="section level2">
<h2><span class="header-section-number">2.1</span> Tweets by day</h2>
<p><img src="figures/bog2019/tweets-by-day-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="tweets-by-day-and-time" class="section level2">
<h2><span class="header-section-number">2.2</span> Tweets by day and time</h2>
<p>Filtered for dates 2019-05-07 - 2019-05-11 in the America/New_York timezone.</p>
<p><img src="figures/bog2019/tweets-by-day-hour-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="users" class="section level1">
<h1><span class="header-section-number">3</span> Users</h1>
<div id="top-tweeters" class="section level2">
<h2><span class="header-section-number">3.1</span> Top tweeters</h2>
<p><img src="figures/bog2019/tweets-top-users-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-original-tweeters" class="section level2">
<h2><span class="header-section-number">3.2</span> Top original tweeters</h2>
<p><img src="figures/bog2019/tweets-top-users-orig-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-proportion" class="section level2">
<h2><span class="header-section-number">3.3</span> Retweet proportion</h2>
<p><img src="figures/bog2019/tweets-retweet-prop-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-time" class="section level2">
<h2><span class="header-section-number">3.4</span> Top tweeters by time</h2>
<p><img src="figures/bog2019/tweets-top-users-time-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-tweeters-by-day" class="section level2">
<h2><span class="header-section-number">3.5</span> Top tweeters by day</h2>
<div id="all-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.1</span> All tweets</h3>
<div id="day-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/bog2019/tweets-top-users-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/bog2019/tweets-top-users-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/bog2019/tweets-top-users-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/bog2019/tweets-top-users-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5" class="section level4 unnumbered">
<h4>Day 5</h4>
<p><img src="figures/bog2019/tweets-top-users-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="original-tweets" class="section level3 tabset">
<h3><span class="header-section-number">3.5.2</span> Original tweets</h3>
<div id="day-1-1" class="section level4 unnumbered">
<h4>Day 1</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day1-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-2-1" class="section level4 unnumbered">
<h4>Day 2</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day2-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-3-1" class="section level4 unnumbered">
<h4>Day 3</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day3-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-4-1" class="section level4 unnumbered">
<h4>Day 4</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day4-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="day-5-1" class="section level4 unnumbered">
<h4>Day 5</h4>
<p><img src="figures/bog2019/tweets-top-users-orig-day5-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
</div>
<div id="sources" class="section level2">
<h2><span class="header-section-number">3.6</span> Sources</h2>
<p><img src="figures/bog2019/tweets-top-sources-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="networks" class="section level1">
<h1><span class="header-section-number">4</span> Networks</h1>
<div id="replies" class="section level2">
<h2><span class="header-section-number">4.1</span> Replies</h2>
<p>The “replies network”, composed from users who reply directly to one another, coloured by page rank.</p>
<p><img src="figures/bog2019/reply-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="mentions" class="section level2">
<h2><span class="header-section-number">4.2</span> Mentions</h2>
<p>The “mentions network”, where users mention other users in their tweets. Node colour, label size and alpha adjusted according to PageRank score.</p>
<p><img src="figures/bog2019/mentions-network-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="retweets" class="section level1">
<h1><span class="header-section-number">5</span> Retweets</h1>
<div id="retweet-proportion-1" class="section level2">
<h2><span class="header-section-number">5.1</span> Retweet proportion</h2>
<p><img src="figures/bog2019/is-retweet-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="retweet-count" class="section level2">
<h2><span class="header-section-number">5.2</span> Retweet count</h2>
<p><img src="figures/bog2019/retweet-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-retweets" class="section level2">
<h2><span class="header-section-number">5.3</span> Top retweets</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
retweet_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Peak east coast techno-hipster cryptocurrency Americana cannabis genome #bog19 <a href="https://t.co/x4hOYHdeGe" class="uri">https://t.co/x4hOYHdeGe</a>
</td>
<td style="text-align:right;">
61
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Just in time for #bog19, <span class="citation">@genomeresearch</span> posts early access version (today) of our paper where we found 2250 bacterial genomes that had human contamination, plus 3,437 “proteins” that are erroneous translations of these human contaminants <a href="https://t.co/koQRkmLRPw" class="uri">https://t.co/koQRkmLRPw</a>
</td>
<td style="text-align:right;">
42
</td>
</tr>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:left;">
athmapai
</td>
<td style="text-align:left;">
Recent or soon to be grads: looking for a postdoc in gene regulation, RNA, and genomics? I’d love to chat! Come find me at #BoG19! I’ll be presenting poster #185 on Thursday <a href="https://t.co/FBsPTVH0Tn" class="uri">https://t.co/FBsPTVH0Tn</a>
</td>
<td style="text-align:right;">
26
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
Do you know how many questions have been asked since the last one asked by a woman? I do. It’s 32. #BoG19
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:left;">
JedMSP
</td>
<td style="text-align:left;">
Oxford Nanopour #bog19 <a href="https://t.co/QeSusqSoaz" class="uri">https://t.co/QeSusqSoaz</a>
</td>
<td style="text-align:right;">
20
</td>
</tr>
<tr>
<td style="text-align:left;">
hagentilgner
</td>
<td style="text-align:left;">
Wow: ScISOr-Seq type work is catching on (<a href="https://t.co/JVU0YLcLLQ" class="uri">https://t.co/JVU0YLcLLQ</a>). Happy to see this and absolutely have to read in detail. BTW, if you like single-cell isoform work (see <a href="https://t.co/OFPcZDaxMt" class="uri">https://t.co/OFPcZDaxMt</a>), come and see me at poster 271. We are hiring! #BoG19
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Anonymous questions from the audience (moderated) are a terrific way to include more audience members. Otherwise the questions tend to be from a small cadre of frequent-askers. Plus it allows people to phrase their questions more carefully #BoG19 <a href="https://t.co/TOFIHMVlog" class="uri">https://t.co/TOFIHMVlog</a>
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
For those of you interested in the experiment we’re doing at #BoG19, here’s some preliminary results (right before the last talk): Participation, <em>in person</em>, has changed drastically. For the first time ever, we’ve accidentally created overrepresentation. <a href="https://t.co/3Zn0JSDFFS" class="uri">https://t.co/3Zn0JSDFFS</a>
</td>
<td style="text-align:right;">
16
</td>
</tr>
<tr>
<td style="text-align:left;">
aaronquinlan
</td>
<td style="text-align:left;">
Really fun talk from <span class="citation">@BeEngelhardt</span> on a Bayesian approach for optimizing experimental design to maximize cell type discovery from single-cell sequencing experiments. Calling it GT-TS (Good-Toulmin like estimator via Thompson sampling). Paper here: <a href="https://t.co/uWm7ySP4lG" class="uri">https://t.co/uWm7ySP4lG</a> #bog19
</td>
<td style="text-align:right;">
14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="favourites" class="section level1">
<h1><span class="header-section-number">6</span> Favourites</h1>
<div id="favourite-proportion" class="section level2">
<h2><span class="header-section-number">6.1</span> Favourite proportion</h2>
<p><img src="figures/bog2019/has-favorite-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="favourite-count" class="section level2">
<h2><span class="header-section-number">6.2</span> Favourite count</h2>
<p><img src="figures/bog2019/favorite-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-favourites" class="section level2">
<h2><span class="header-section-number">6.3</span> Top favourites</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Peak east coast techno-hipster cryptocurrency Americana cannabis genome #bog19 <a href="https://t.co/x4hOYHdeGe" class="uri">https://t.co/x4hOYHdeGe</a>
</td>
<td style="text-align:right;">
231
</td>
</tr>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
203
</td>
</tr>
<tr>
<td style="text-align:left;">
JedMSP
</td>
<td style="text-align:left;">
Oxford Nanopour #bog19 <a href="https://t.co/QeSusqSoaz" class="uri">https://t.co/QeSusqSoaz</a>
</td>
<td style="text-align:right;">
152
</td>
</tr>
<tr>
<td style="text-align:left;">
jtung5
</td>
<td style="text-align:left;">
At the #BoG19 closing banquet and the big news is that <span class="citation">@blekhman</span> just received tenure!! Big congrats Ran, so well deserved!! <a href="https://t.co/9s8c9bMgdX" class="uri">https://t.co/9s8c9bMgdX</a>
</td>
<td style="text-align:right;">
117
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Just in time for #bog19, <span class="citation">@genomeresearch</span> posts early access version (today) of our paper where we found 2250 bacterial genomes that had human contamination, plus 3,437 “proteins” that are erroneous translations of these human contaminants <a href="https://t.co/koQRkmLRPw" class="uri">https://t.co/koQRkmLRPw</a>
</td>
<td style="text-align:right;">
82
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Anonymous questions from the audience (moderated) are a terrific way to include more audience members. Otherwise the questions tend to be from a small cadre of frequent-askers. Plus it allows people to phrase their questions more carefully #BoG19 <a href="https://t.co/TOFIHMVlog" class="uri">https://t.co/TOFIHMVlog</a>
</td>
<td style="text-align:right;">
79
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
Do you know how many questions have been asked since the last one asked by a woman? I do. It’s 32. #BoG19
</td>
<td style="text-align:right;">
76
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
#bog19 the first BOG without the portrait of Watson. Finally. <a href="https://t.co/i42jfyMKdt" class="uri">https://t.co/i42jfyMKdt</a>
</td>
<td style="text-align:right;">
63
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
For those of you interested in the experiment we’re doing at #BoG19, here’s some preliminary results (right before the last talk): Participation, <em>in person</em>, has changed drastically. For the first time ever, we’ve accidentally created overrepresentation. <a href="https://t.co/3Zn0JSDFFS" class="uri">https://t.co/3Zn0JSDFFS</a>
</td>
<td style="text-align:right;">
60
</td>
</tr>
<tr>
<td style="text-align:left;">
StevenSalzberg1
</td>
<td style="text-align:left;">
Love it that the genome scaffolders used by <span class="citation">@mike_schatz</span> for the tomato genome are called SALSA2 and RaGOO. Coincidence? #BoG19
</td>
<td style="text-align:right;">
60
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="quotes" class="section level1">
<h1><span class="header-section-number">7</span> Quotes</h1>
<div id="quote-proportion" class="section level2">
<h2><span class="header-section-number">7.1</span> Quote proportion</h2>
<p><img src="figures/bog2019/is-quote-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="quote-count" class="section level2">
<h2><span class="header-section-number">7.2</span> Quote count</h2>
<p><img src="figures/bog2019/quotes-count-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-quotes" class="section level2">
<h2><span class="header-section-number">7.3</span> Top quotes</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
quote_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
AnnaCuomo92
</td>
<td style="text-align:left;">
Great Illustration and great meeting so far! #BoG19 <a href="https://t.co/MbRJgVUS5E" class="uri">https://t.co/MbRJgVUS5E</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
Awesomics
</td>
<td style="text-align:left;">
The Biology of Genomes (#BoG19) now underway <a href="https://t.co/uJpygePI9e" class="uri">https://t.co/uJpygePI9e</a>. Access live streamed talks, abstract PDF, and video archive (for 6 weeks) at <a href="https://t.co/L8JiHY0BFH" class="uri">https://t.co/L8JiHY0BFH</a> and enter “99” in the ‘Payment Authorized’ field. Cost=$99. Thanks <span class="citation">@cshlmeetings</span>! <a href="https://t.co/a4zUxtCduV" class="uri">https://t.co/a4zUxtCduV</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
deboramarks
</td>
<td style="text-align:left;">
Club #BoG19 ever wants me as a ‘member’ I’d pretend I didn’t want to join but then I’d say yes within 30 seconds-sounds like a great conf <a href="https://t.co/8hW8w9NCcf" class="uri">https://t.co/8hW8w9NCcf</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
mammothbiosci
</td>
<td style="text-align:left;">
Exciting to hear about everything going on at #BoG19 - work in #genome #biology helps drive the creation of DNA-based diagnostic tools. <a href="https://t.co/9kf9P9T051" class="uri">https://t.co/9kf9P9T051</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
DevilsAdvoMax
</td>
<td style="text-align:left;">
Big, big thanks <span class="citation">@NatalieTelis</span> and Friday chairs for pushing towards participatory diversity in the QAs. Clear improvements over the course of #BoG19 sessions - a more inviting spirit for young trainees, first-time attendees or simply more hesitant folks. Exemplary activism! 💕🙂 <a href="https://t.co/G1rrlNS8bB" class="uri">https://t.co/G1rrlNS8bB</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
<tr>
<td style="text-align:left;">
girlscientist
</td>
<td style="text-align:left;">
Interesting – here’s what offering the chance to ask questions online during a talk did to in-person questions at #BoG19 (thread)…. <a href="https://t.co/zzr6KPoOer" class="uri">https://t.co/zzr6KPoOer</a>
</td>
<td style="text-align:right;">
2
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="media" class="section level1">
<h1><span class="header-section-number">8</span> Media</h1>
<div id="media-count" class="section level2">
<h2><span class="header-section-number">8.1</span> Media count</h2>
<p><img src="figures/bog2019/has-media-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="top-media" class="section level2">
<h2><span class="header-section-number">8.2</span> Top media</h2>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
favorite_count
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
Peak east coast techno-hipster cryptocurrency Americana cannabis genome #bog19 <a href="https://t.co/x4hOYHdeGe" class="uri">https://t.co/x4hOYHdeGe</a>
</td>
<td style="text-align:right;">
231
</td>
</tr>
<tr>
<td style="text-align:left;">
MollyGasp
</td>
<td style="text-align:left;">
Like this year’s #BoG19 cover illustration? You can thank the multi-talented <span class="citation">@Anna_Minkina</span>, a talented artist and an even-more-talented grad student in the <span class="citation">@JShendure</span> lab! <a href="https://t.co/1KGplQaqSL" class="uri">https://t.co/1KGplQaqSL</a>
</td>
<td style="text-align:right;">
203
</td>
</tr>
<tr>
<td style="text-align:left;">
JedMSP
</td>
<td style="text-align:left;">
Oxford Nanopour #bog19 <a href="https://t.co/QeSusqSoaz" class="uri">https://t.co/QeSusqSoaz</a>
</td>
<td style="text-align:right;">
152
</td>
</tr>
<tr>
<td style="text-align:left;">
jtung5
</td>
<td style="text-align:left;">
At the #BoG19 closing banquet and the big news is that <span class="citation">@blekhman</span> just received tenure!! Big congrats Ran, so well deserved!! <a href="https://t.co/9s8c9bMgdX" class="uri">https://t.co/9s8c9bMgdX</a>
</td>
<td style="text-align:right;">
117
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
#bog19 the first BOG without the portrait of Watson. Finally. <a href="https://t.co/i42jfyMKdt" class="uri">https://t.co/i42jfyMKdt</a>
</td>
<td style="text-align:right;">
63
</td>
</tr>
<tr>
<td style="text-align:left;">
NatalieTelis
</td>
<td style="text-align:left;">
For those of you interested in the experiment we’re doing at #BoG19, here’s some preliminary results (right before the last talk): Participation, <em>in person</em>, has changed drastically. For the first time ever, we’ve accidentally created overrepresentation. <a href="https://t.co/3Zn0JSDFFS" class="uri">https://t.co/3Zn0JSDFFS</a>
</td>
<td style="text-align:right;">
60
</td>
</tr>
<tr>
<td style="text-align:left;">
aaronquinlan
</td>
<td style="text-align:left;">
Are you interested in what can be learned about germline mutation dynamics from WGS of large, 3-generation pedigrees? Check out <span class="citation">@tomsasani</span>’s poster (240) today. Tom is exceptional and is looking for postdoc opps in the coming year. Preprint: <a href="https://t.co/CwpBnGP0f1" class="uri">https://t.co/CwpBnGP0f1</a> #bog19 <a href="https://t.co/DkbX1KqHPh" class="uri">https://t.co/DkbX1KqHPh</a>
</td>
<td style="text-align:right;">
58
</td>
</tr>
<tr>
<td style="text-align:left;">
kajawasik
</td>
<td style="text-align:left;">
Whenever I’m back at <span class="citation">@CSHL</span> I realize how much I miss this place. It really made me a scientist and taught me to enjoy the small things in life like wine and cheese on President’s lawn.. #bog19 <a href="https://t.co/efXS121Mh9" class="uri">https://t.co/efXS121Mh9</a>
</td>
<td style="text-align:right;">
46
</td>
</tr>
<tr>
<td style="text-align:left;">
NeBanovich
</td>
<td style="text-align:left;">
Fish mafia #bog19 #GenomesSparkJoy <a href="https://t.co/yEgLgcQlAx" class="uri">https://t.co/yEgLgcQlAx</a>
</td>
<td style="text-align:right;">
40
</td>
</tr>
<tr>
<td style="text-align:left;">
notSoJunkDNA
</td>
<td style="text-align:left;">
#bog19 “Germline and somatic variant calling with NovaSeq™ 6000 2x250bp reads” <span class="citation">@nygenome</span> poster with <span class="citation">@mjs2225</span> <span class="citation">@gnarzisi</span> et al. To be presented on Friday but available at <a href="https://t.co/tETp7Brcmz" class="uri">https://t.co/tETp7Brcmz</a> <a href="https://t.co/Xky5B00oXy" class="uri">https://t.co/Xky5B00oXy</a>
</td>
<td style="text-align:right;">
34
</td>
</tr>
</tbody>
</table>
<div id="most-liked-media-image" class="section level3">
<h3><span class="header-section-number">8.2.1</span> Most liked media image</h3>
<div class="figure">
<img src="http://pbs.twimg.com/media/D6Je2pUXoAUjpUq.jpg" />

</div>
</div>
</div>
</div>
<div id="tweet-text" class="section level1">
<h1><span class="header-section-number">9</span> Tweet text</h1>
<div id="word-cloud" class="section level2">
<h2><span class="header-section-number">9.1</span> Word cloud</h2>
<p>The top 100 words used 3 or more times.</p>
<p><img src="figures/bog2019/count-words-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="bigram-graph" class="section level2">
<h2><span class="header-section-number">9.2</span> Bigram graph</h2>
<p>Words that were tweeted next to each other at least 5 times.</p>
<p><img src="figures/bog2019/bigram-graph-1.png" width="960" style="display: block; margin: auto;" /></p>
</div>
<div id="topic-modelling" class="section level2">
<h2><span class="header-section-number">9.3</span> Topic modelling</h2>
<p>Top 10 words associated with 6 topics identified by LDA.</p>
<p><img src="figures/bog2019/topics-words-1.png" width="960" style="display: block; margin: auto;" /></p>
<div id="representative-tweets" class="section level3 tabset">
<h3><span class="header-section-number">9.3.1</span> Representative tweets</h3>
<p>Most representative tweets for each topic</p>
<div id="topic-1" class="section level4 unnumbered">
<h4>Topic 1</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
AH: Performed matched unrelated-GWAS vs sib-GWAS to look at this “three headed dragon”. Can plot sib-GWAS vs unrelated. Thankfully physical traits line up pretty well, but social traits (educational attainment, smoking, age at first sex) show <em>strong</em> shifts #bog19
</td>
<td style="text-align:right;">
0.9935852
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
DV: Gene pathway connectivity? Assign genes to 83% of variants. Want to identify the core genes (under omnigenic model). For blood cell traits, obvious to choose mendelian genes. Then use co-expression network in whole blood to see if mendelian genes behave like core genes #BoG19
</td>
<td style="text-align:right;">
0.9935852
</td>
</tr>
<tr>
<td style="text-align:left;">
rnomics
</td>
<td style="text-align:left;">
<p>Ewan Birney on Twitter: “Sek takes us through the construction of polygenic risk score - from MI GWAS, selected best score from 20 with first round of UK BioBank, then test on remainder. PRS is not correlated to the ACC/AHA risk score #BoG19” <a href="https://t.co/iN0JYt345f" class="uri">https://t.co/iN0JYt345f</a></p>
twitte…
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
SM: acquired genomic mosaicism most apparent in blood-derived DNA (small number of SNPs that have unusual small allelic fraction in exome sequencing). Clonal hematopoiesis rarely detected in young individual, process takes decades to make up sig fraction of cell pop. #BoG19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
AH: what to investigate if other factors beyond ancestry impacts PS. Used the “White British” of UKB. Performed GWAS in 120K men + 120K women, and then tested prediction accuracy in 20K men and 20K women, separately. Prediction accuracy differed by sex. #BoG19
</td>
<td style="text-align:right;">
0.9930763
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Coronary artery disease is the leading global cause of mortality</p>
<p>Current risk assessment of heart attack reviewed by Khera et al <a href="https://t.co/7BJCbpN2xv" class="uri">https://t.co/7BJCbpN2xv</a> based on non-🧬metrics</p>
but <span class="citation">@skathire</span> has now shown that polygenic risk score is the first risk factor for this disease #BoG19 <a href="https://t.co/UuHPODF7cq" class="uri">https://t.co/UuHPODF7cq</a>
</td>
<td style="text-align:right;">
0.9930763
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
AH: under null, prediction accuracy should be equal between GWAS and sib-GWAS. If there are indirect effects, GWAS-based PS should outperform sib-GWAS-based. Find for most traits standard GWAS outperforms (matched sample size between GWASes) #BoG19
</td>
<td style="text-align:right;">
0.9930763
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Peter <span class="citation">@genemodeller</span> on types of joint analysis:</p>
<ul>
<li>meta-analysis</li>
<li>fine-mapping</li>
<li>PheWAS-analysis</li>
</ul>
<p>You could also cluster GWAS results. Did so using a bayesian approach (would make <span class="citation">@BeEngelhardt</span> proud ^^) based on a Dirichlet prior</p>
<p>Can analyze simultaneously &amp;gt;1k GWAS</p>
#BoG19
</td>
<td style="text-align:right;">
0.9930763
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Jerber/2: inducing dopaminergic neurons (3 time points, 11-30-52 days) and perturbations (toxicity drugs: rotenone); from 129 donors; Capturing nicely the cell types. Rotenone treatment shows clear gene expression differences -&amp;gt; discovers eQTLs for all cell types. #BoG19
</td>
<td style="text-align:right;">
0.9930763
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
SK: Hypothesis = polygenic scores including genome-wide set of SNPs can identify more individuals at risk of MI. 60k cases vs. 120k controls, GWAS with imputation data to construct PRS, and then validated PRS in UK Biobank. #BoG19
</td>
<td style="text-align:right;">
0.9930763
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-2" class="section level4 unnumbered">
<h4>Topic 2</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
majbvc
</td>
<td style="text-align:left;">
El <span class="citation">@centrodemocratico</span> buscará un candidato distinto a Ángela Garzón para enfrentar las elecciones en Bogotá. Será…??? O será otra salida en falso…?? #losleo <span class="citation">@AlvaroUribeVel</span> <span class="citation">@diegomolanovega</span> <span class="citation">@SamuelHoyosM</span> #FelizJueves #BoG19 #filbo2019 #video <span class="citation">@ChurchillBogota</span> <span class="citation">@MiguelUribeT</span> <a href="https://t.co/GUJ8jaznOy" class="uri">https://t.co/GUJ8jaznOy</a>
</td>
<td style="text-align:right;">
0.9940245
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
AS: studying at single cell level. scTRIP - explore structural variation cell by cell. Culture cell, add BrdU so can differentiate each sister chromatids during mitosis. So sequence each strand, and map to human reference genome to determine whether crick or watson strand. #BoG19
</td>
<td style="text-align:right;">
0.9938126
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Joseph Thornton used an ordinal logistic regression model to study epistasis effects and main effects</p>
<p>A joint model includes 2 extra sum of coefficients (sum of i to j)</p>
<p>Misclassification rate is &amp;lt; 0.6% overall</p>
<p>Presented a heatmap of main vs epistasis significant terms</p>
#BoG19
</td>
<td style="text-align:right;">
0.9935852
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Bock/2. One approach is ATAC-seq of CLL, time points, and 43K single cell transcriptomics to identify open sites and look back at responsible sites; and use the data to reconstruct response to ibrutinib in CLL. Single cell helpful to prioritize drug treatments #bog19 <span class="citation">@erikarner</span>
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Elise <span class="citation">@EliseScience</span> thinks that the + and - correlated eQTLs with aFC could be due to transcription factor binding levels</p>
<p>Correlated aFC with TF expression in each tissue (used ENCODE data)</p>
<p>5.6% of tested TF-eQTLs pairs are corr with 1 TF 41% of are corr with &amp;gt; 1TF</p>
#BoG19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
sorrywm
</td>
<td style="text-align:left;">
.<span class="citation">@mikkelschierup</span> uses a “recombination clock” (distribution of introgressed archaic segment lengths) to estimate # of generations since archaic admixture. Segment lengths correlate w/# derived variants from mutation types assoc. with paternal age. -&amp;gt; Diff. generation times #BoG19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Bock/3 patient specific “route planning” on disease landscape. Beyond landscape: very high throughput CRISPRs for causality; No pooled, no arrayed CRISPRs, but single cell, linking guide RNA back to transcriptome. Use a perturb-seq vector and modified technology. #bog19
</td>
<td style="text-align:right;">
0.9930763
</td>
</tr>
<tr>
<td style="text-align:left;">
obahcall
</td>
<td style="text-align:left;">
Arnav Moudgil <span class="citation">@arnavm1</span> presents another single cell combo method, now for TF. Single cell calling cards (scCC) method combines self-reporting transposons + scRNAseq, to map TF binding sites together with expression in single cells. #BoG19
</td>
<td style="text-align:right;">
0.9927903
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Mikkel <span class="citation">@mikkelschierup</span> is talking about Skov et al, bioRxiv, 2018 <a href="https://t.co/modrYeXWWz" class="uri">https://t.co/modrYeXWWz</a></p>
<p>They can use this to derive mean archaic segmenting lengths (SGDP) &amp;amp; the total amount of archaic sequence</p>
<p>Then they explored how the segments are shared by West Eurasia &amp;amp; East Asia</p>
#BoG19 <a href="https://t.co/XSh7Ah1kmI" class="uri">https://t.co/XSh7Ah1kmI</a>
</td>
<td style="text-align:right;">
0.9927903
</td>
</tr>
<tr>
<td style="text-align:left;">
sorrywm
</td>
<td style="text-align:left;">
EWong uses patterns of synteny and text scanning based on “bags of motifs” for TFs to find equivalent enhancers across highly divergent (unalignable) species. As a proof of concept, she uses enhancers from sponge to drive expression in zebrafish and mouse. Beautiful work. #BoG19
</td>
<td style="text-align:right;">
0.9927903
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-3" class="section level4 unnumbered">
<h4>Topic 3</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
JX: patients hospitalized for bone marrow transplantation provide longitudinal data for host-microbiome studies. For ~2 weeks patients are immunocompromised, hence carefully monitored. Has a lot of clinical variable/meta-data, and can collect stool sample for microbiome. #BoG19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
HM: male-bias not explained by de novo variants found on X chr. Haldane proposed a theory of de novo vs. inherited pathogenic variants in X-linked recessive genes, based on mutation-sel balanec. Expect 18%, but observed 32% #BoG19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Julie <span class="citation">@LiliNYC11</span> focused on dopaminergic neurons from substantia nigra, related to Parkinson’s disease</p>
<p>Built HipSci: nice logo!</p>
<p>Ran into some problems a few times and had to start from a frozen pool. Ideally, start from a fresh sample</p>
Cells followed expected patt day 11 #BoG19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
KL: Channel island foxes, population size ~300-2000; S. Nicolas foxes severely reduced diversity across the genome, &amp;gt; 10x reduction in heterozygosity. Adding more samples, including samples from the past (1929), and low diversity has been observed even in old samples. #BoG19
</td>
<td style="text-align:right;">
0.9930763
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Hannah Currant, on UK Biobank, 500K individuals, 96M variants: focus on retinal phenotype. Retina inspected for physical differences (example: thickness, as “endophenotype”), which is connected with diseases/biology. ~31K GWAS- identify loci associated with retina problems #bog19
</td>
<td style="text-align:right;">
0.9930763
</td>
</tr>
<tr>
<td style="text-align:left;">
obahcall
</td>
<td style="text-align:left;">
Appreciation for our wonderful #BoG19 organizers, who are also illustrious scientists celebrating recent well deserved recognitions!<br />
Matt Hurles <span class="citation">@mehurles</span> elected FRS <span class="citation">@royalsociety</span> Elaine Ostrander elected <span class="citation">@theNASciences</span> Jay Shendure <span class="citation">@JShendure</span> Lounsbery award <span class="citation">@theNASciences</span>
</td>
<td style="text-align:right;">
0.9927903
</td>
</tr>
<tr>
<td style="text-align:left;">
ewanbirney
</td>
<td style="text-align:left;">
KL: Flipping to Isle Royale population, where they have been recently isolated - very bumpy hetreozygosity plot, and derived deleterious alleles, and not a healthy population. Suggests the Island fox population inbreed with strong deleterious alleles removed #bog19
</td>
<td style="text-align:right;">
0.9927903
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
JMR: exploratory PCA using archaic samples show Denisovan cline for Oceanian population. Neandertal anc ~uniformly distributed (~2-3%), but Denisovan anc more variable (0-3%), and scale well with Papuan ancestry (except some outlier population like Agta HG in Phillipines. #BoG19
</td>
<td style="text-align:right;">
0.9927903
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Valentina Snetkova- ultraconserved sequences and enhancer function (&amp;gt;200nt 100% conservation human/mouse/rat; and beyond). Used in a reporter assay for enhancers, including mutated isoforms. Assayed in mouse embryos, often in CNS. 78 alleles, 2000 mutations. #bog19
</td>
<td style="text-align:right;">
0.9927903
</td>
</tr>
<tr>
<td style="text-align:left;">
sorrywm
</td>
<td style="text-align:left;">
JXavier finds that gut microbial diversity is predictive of recovery from bone marrow transplants, uses modeling in model systems to identify inter-microbial interactions, and tries out recovering a patient’s microbiome using their own pre-operation fecal sample. #BoG19
</td>
<td style="text-align:right;">
0.9924797
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-4" class="section level4 unnumbered">
<h4>Topic 4</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
JavierHerrero7
</td>
<td style="text-align:left;">
.<span class="citation">@nonfiniteloop</span> — Can we detect post zygotic variations. Re-purpose GTEx data. Develop method to detect mutations from bulk RNA-seq and validate in Sun exposed/unexposed skin. More mutations in UV exposed skin, increase in UV-induced mut. signature #BoG19
</td>
<td style="text-align:right;">
0.9940245
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>EG is using 12 pcw (post conception weeks) human neural stem cells with a CRISPR/Cas9 deletion screen to learn more about enhancer function (2,233 of them) across 26,260 non-coding regions and 10,656 coding regions expressed in this cell line, + 2,608 bkg regs via ROADMAP</p>
#BoG19
</td>
<td style="text-align:right;">
0.9940245
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Sanders: Structural variations in single cells. Developed a method for single cell genome sequencing which maintains strand information, taking advantage of BdrU incorporation -&amp;gt; digestion of one strand -&amp;gt; sequencing the remaining strand. Measures depth, strand, phase. #BoG19
</td>
<td style="text-align:right;">
0.9940245
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Ashley Sanders developed Strand-seq which takes advantage of the paternal homologues to track cells across divisions</p>
<p><a href="https://t.co/K3xs3OcG5I" class="uri">https://t.co/K3xs3OcG5I</a> abstract: “resolves the individual homologs within a cell”</p>
<p>scTRIP: single cell Tri-channel processing is the follow up method</p>
#BoG19 <a href="https://t.co/jKG5JgFcfW" class="uri">https://t.co/jKG5JgFcfW</a>
</td>
<td style="text-align:right;">
0.9935852
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Javier Mendoza-Revilla, history of south-west Pacific migrations. Only a recent 5.5 k years ago went to far Oceania, then replaced by Papuan ancestry 2.5K years ago. Complex -&amp;gt; sequences 317 WHG. PCA separates nicely for geographical regions of donors. #BoG19
</td>
<td style="text-align:right;">
0.9935852
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Daniel <span class="citation">@vervacity</span> clustered ATAC peaks to find dynamic accessible regions, then looked into H3K27ac and H3K4me1 data</p>
<p>Lead to the question: how does DNA sequence lead to these molecular phenotypes? (the dynamic regions)</p>
<p>Goal: use DNA sequence to predict chromatin activ.</p>
#bog19
</td>
<td style="text-align:right;">
0.9935852
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
JMR: Settlement of Near / Remote Oceania - modern humans arrived in Near Oceania ~40-65kya, but did not extend beyond Soloman island. Austronesian expansion from Taiwan ~ 5.5 kya; Lapita people arrived in Vanuatu ~3kya; mirgration from Bismarck Archipelago ~2.5kya. #BoG19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>YQ from <span class="citation">@MarthGabor</span> wants to study subclone-specific mutations, #scRNAseq fits the task very well</p>
<p>Challenge: cell coverage is low, roughly 3 MB of “real-estate” for determining the presence of somatic variants, so only ~20 somatic variants with coverage</p>
<ul>
<li>allele drop-out #BoG19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Julie Jerber, mapping neuronal development eQTLs. eQTL limitations: tissue, time points, stimulations not feasible (in vivo). Instead use hiPS cells. Go beyond primary blood cells! using iPS can stimulate and then analyze at single cell level (capture different cells). #BoG19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Luca Pinello <span class="citation">@lucapinello</span> CRISPResso (cool name!): analysis of CRISPR-Cas genome editing outcomes from deep sequencing! Now CRISPResso2! Which will take into account base modification, batch mode, allele specific quantification, biology informed alignment, fast processing! #bog19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
</tbody>
</table></li>
</ul>
</div>
<div id="topic-5" class="section level4 unnumbered">
<h4>Topic 5</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Marina Ruiz-Romero (cc <span class="citation">@GuigoLab</span>) has ~50 million Illumina TruSeq 75 bp reads from 9 fly samples across 3 developmental stages</p>
<p>She found that tissue specificity increases along differentiation from larva 3 to white pupa to fly (?)</p>
<p>matches modEncode data based on PC1/2</p>
#BoG19
</td>
<td style="text-align:right;">
0.9935852
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Now Hanna Ollila <span class="citation">@hannamollila</span> from <span class="citation">@jkpritch</span>’s lab (his lab meetings must feel like today’s morning session hehe) on USF1 as a pleiotropic meta regular <em>for sleep and circadian rhythms</em></p>
<p>Story started back in her PhD in Helsinki</p>
<p><a href="https://t.co/cVT9ZwKbbc" class="uri">https://t.co/cVT9ZwKbbc</a></p>
#BoG19
</td>
<td style="text-align:right;">
0.9930763
</td>
</tr>
<tr>
<td style="text-align:left;">
PLOSBiology
</td>
<td style="text-align:left;">
John Wolters <span class="citation">@HittingerLab</span>: Engineering yeast mtDNA: deplete mtDNA in recipient, put manipulated mtDNA back with gene gun, prime translation, go! Use small, less AT-rich Naumovozyme castellii backbone with S. cerevisiae genes patched in. Ongoing… #BoG19
</td>
<td style="text-align:right;">
0.9924797
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Project has simple design <span class="citation">@mike_schatz</span> : select diverse samples, sequence, find SVs, … (missed the rest, sry!)</p>
<p>Optimized sample selection uses SVCollector by <span class="citation">@sedlazeck</span> et al <a href="https://t.co/Qxw3dhsTSw" class="uri">https://t.co/Qxw3dhsTSw</a></p>
<p>PromethION by <span class="citation">@nanopore</span> was a game changer for the proj, lots of 🧬</p>
#BoG19 <a href="https://t.co/OW5a1DezzD" class="uri">https://t.co/OW5a1DezzD</a>
</td>
<td style="text-align:right;">
0.9924797
</td>
</tr>
<tr>
<td style="text-align:left;">
ryanlayer
</td>
<td style="text-align:left;">
Children’s Hospital Colorado Precision Diagnostics team is hiring a Bioinformatic Software Engineer. If you want to help diagnose sick kids by improving the speed and accuracy of NGS algorithms DM me or find me at #BoG19 #bioinformatics #dev #jobs
</td>
<td style="text-align:right;">
0.9921411
</td>
</tr>
<tr>
<td style="text-align:left;">
ficalvert
</td>
<td style="text-align:left;">
Thought I’d beaten jet lag by forcing myself to stay up late on my first night in New York. My 5am wake up this morning was a brutal reminder that you can never beat jet lag! Excited to go explore the city after using my early start to work on my #BoG19 slides!
</td>
<td style="text-align:right;">
0.9921411
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
TR: nearly 1M beef cattle animal genotyped. Now have large accurately imputed dataset, temporally distributed tracing back to specimen from 70s. Recent history well understood (pedigree to early 1900s), strong and recent selection. #BoG19
</td>
<td style="text-align:right;">
0.9921411
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Oh! Marina Ruiz-Romero used BrainSpan data (Kang et al, 2011 <a href="https://t.co/f0gItoBnX5" class="uri">https://t.co/f0gItoBnX5</a>)</p>
<p>I didn’t see that coming! However, in retrospect it makes sense. Marina compared fly development to mouse development, then used BrainSpan to look at conservation in human dev</p>
#BoG19 <span class="citation">@GuigoLab</span>
</td>
<td style="text-align:right;">
0.9921411
</td>
</tr>
<tr>
<td style="text-align:left;">
sorrywm
</td>
<td style="text-align:left;">
Tasty lager generation is indeed a convergent trait in yeast, and JWolters paints a much more aesthetically pleasing image of this on the tree than I managed in my slides. Raising my glass to the amazing work of the <span class="citation">@HittingerLab</span> on generating genomic resources for yeast. #BoG19
</td>
<td style="text-align:right;">
0.9921411
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>Super cool slide by <span class="citation">@TroyNRowan</span> showing the changes in genotype frequencies over time (longitudinal genetics 😱🤩) on loci related to horns (POLLED), color and… (forgot it :P) in US beef cattle 🐄</p>
<p>A super popular 2012 semental leads registrations</p>
#BoG19 <a href="https://t.co/vr77vHtbmW" class="uri">https://t.co/vr77vHtbmW</a>
</td>
<td style="text-align:right;">
0.9921411
</td>
</tr>
</tbody>
</table>
</div>
<div id="topic-6" class="section level4 unnumbered">
<h4>Topic 6</h4>
<table>
<thead>
<tr>
<th style="text-align:left;">
screen_name
</th>
<th style="text-align:left;">
text
</th>
<th style="text-align:right;">
gamma
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
carninci
</td>
<td style="text-align:left;">
Farren Isaacs; Multiple automated genome engineering (MAGE) works without double strand brake (big advantage). Eukaryotic MAGE. Using a ssODN annealing DNA replication fork enhances gene editing 100x. Multiplexing: msh2- allow multiple editing events. Cycling adds variants #bog19
</td>
<td style="text-align:right;">
0.9942223
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
RJ: Causal effect size distribution parameterized by effect size (beta) and number of causal SNPs (k) (Zhang et al. NG 2018). Want to know how is heritability divided - centered on 1 SNP or distributed among multiple SNPs, has implication to effect distr at regional level. #BoG19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
fellgernon
</td>
<td style="text-align:left;">
<p>SMAC-seq by Georgi Marinov and team re-captures the peaks by DNA-seq peak but also overlaps H4S37C chemical nucleosome mapping peaks</p>
<p>Their profiles are nicely phased with Cse4 ChIP-seq as well</p>
<p>(I don’t know what Cse4 nor H4S37C do! Tweet and later revisit? :P)</p>
#bog19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
RB: 16S microbiome - 95 common taxa, plus microbiome PCs and richness measures. Heritability model controls for age, sex, social interaction network, maternal effect, technical effect, environmental effects (diet, rainfall, season)… #BoG19
</td>
<td style="text-align:right;">
0.9933405
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
BP: observation “SNP with lower levels of LD have significantly larger per-SNP heritability” - MAF/LD-dependent architecture biases SNP heritability estimation. Today will discuss assumptions of estimation, and how to drop them to estimate SNP h2. #BoG19
</td>
<td style="text-align:right;">
0.9930763
</td>
</tr>
<tr>
<td style="text-align:left;">
cshlmeetings
</td>
<td style="text-align:left;">
Unable to make it to next week’s #BoG19? Live stream it! Each $99 subscription includes live stream of talks, PDF of the abstract book, and six-week access to #BoG19’s video archive. Register online: <a href="https://t.co/fMcgEzbvys" class="uri">https://t.co/fMcgEzbvys</a> and enter “99” in the ‘Payment Authorized’ field. <a href="https://t.co/AHcQxt1Sg6" class="uri">https://t.co/AHcQxt1Sg6</a>
</td>
<td style="text-align:right;">
0.9927903
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
RJ: BEAVR estimates local polygenicity from GWAS summary statistics. Use GWAS “noisy” effect size estimates to infer underlying effect size distr. Can estimate # of causal SNPs across varying level of heritability. #BoG19
</td>
<td style="text-align:right;">
0.9927903
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
BP: per causal h2 can be uniform, category-specific, functionally annotated, or MAF/LD dependent. Here propose generalized random effect (GRE) model - let every SNP in the genome have its own variance, just a constant effect size per SNP. #BoG19
</td>
<td style="text-align:right;">
0.9927903
</td>
</tr>
<tr>
<td style="text-align:left;">
CharlestonCWKC
</td>
<td style="text-align:left;">
DV: annotated hits with clinvar and HGMD - 18 rare known pathogenic variants were identified. In general, pathogenic missense show stronger median effect size and longer tail. Missense of unknown pathogencity or unreported missense all generally have larger effect than syn #BoG19
</td>
<td style="text-align:right;">
0.9927903
</td>
</tr>
<tr>
<td style="text-align:left;">
NatureEcoEvo
</td>
<td style="text-align:left;">
<p>A taste of recent pop gen papers published in NEE:</p>
<p>Admixture with a ghost lineage in bonobos <a href="https://t.co/YGP2Ci0P9I" class="uri">https://t.co/YGP2Ci0P9I</a></p>
<p>Using selection at linked sites to estimate adaptation <a href="https://t.co/RRn3kmVmju" class="uri">https://t.co/RRn3kmVmju</a></p>
Ancient genomes of Neolithic Britain <a href="https://t.co/OXJwkj7hbn" class="uri">https://t.co/OXJwkj7hbn</a> #BoG19
</td>
<td style="text-align:right;">
0.9924797
</td>
</tr>
</tbody>
</table>
<hr />
</div>
</div>
</div>
</div>
<div id="session-info" class="section level1 unnumbered">
<h1>Session info</h1>
<pre><code>## R version 3.5.1 (2018-07-02)
## Platform: x86_64-apple-darwin15.6.0 (64-bit)
## Running under: macOS  10.14.4
## 
## Matrix products: default
## BLAS: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRblas.0.dylib
## LAPACK: /Library/Frameworks/R.framework/Versions/3.5/Resources/lib/libRlapack.dylib
## 
## locale:
## [1] en_GB.UTF-8/en_GB.UTF-8/en_GB.UTF-8/C/en_GB.UTF-8/en_GB.UTF-8
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] bindrcpp_0.2.2     here_0.1           knitr_1.20        
##  [4] viridis_0.5.1      viridisLite_0.3.0  ggraph_1.0.2      
##  [7] wordcloud_2.6      RColorBrewer_1.1-2 ggrepel_0.8.0     
## [10] ggplot2_3.0.0      igraph_1.2.2       topicmodels_0.2-8 
## [13] tidytext_0.2.0     forcats_0.3.0      lubridate_1.7.4   
## [16] tidyr_0.8.1        dplyr_0.7.6        rtweet_0.6.8      
## 
## loaded via a namespace (and not attached):
##  [1] Rcpp_0.12.19      lattice_0.20-35   utf8_1.1.4       
##  [4] assertthat_0.2.0  rprojroot_1.3-2   digest_0.6.18    
##  [7] ggforce_0.1.3     slam_0.1-45       R6_2.3.0         
## [10] plyr_1.8.4        backports_1.1.2   stats4_3.5.1     
## [13] evaluate_0.12     highr_0.7         httr_1.3.1       
## [16] pillar_1.3.0      rlang_0.2.2       lazyeval_0.2.1   
## [19] curl_3.2          Matrix_1.2-14     rmarkdown_1.10   
## [22] labeling_0.3      stringr_1.3.1     munsell_0.5.0    
## [25] broom_0.5.0       compiler_3.5.1    janeaustenr_0.1.5
## [28] pkgconfig_2.0.2   htmltools_0.3.6   openssl_1.0.2    
## [31] tidyselect_0.2.5  tibble_1.4.2      gridExtra_2.3    
## [34] fansi_0.4.0       crayon_1.3.4      withr_2.1.2      
## [37] MASS_7.3-50       SnowballC_0.5.1   grid_3.5.1       
## [40] nlme_3.1-137      jsonlite_1.5      gtable_0.2.0     
## [43] magrittr_1.5      units_0.6-1       scales_1.0.0     
## [46] tokenizers_0.2.1  cli_1.0.1         stringi_1.2.4    
## [49] reshape2_1.4.3    farver_1.0        NLP_0.2-0        
## [52] xml2_1.2.0        tools_3.5.1       glue_1.3.0       
## [55] tweenr_1.0.0      purrr_0.2.5       parallel_3.5.1   
## [58] yaml_2.2.0        tm_0.7-6          colorspace_1.3-2 
## [61] bindr_0.1.1       modeltools_0.2-22</code></pre>
</div>



</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
